  INFO root [2017-11-06 14:24:10,155] - 
  INFO root [2017-11-06 14:24:10,156] - -----------------------------------------------------------------------------------------


  INFO root [2017-11-06 14:24:10,156] - Resmgr::initialize: KBE_ROOT=C:/Users/hyc/Desktop/kbengine/
  INFO root [2017-11-06 14:24:10,156] - Resmgr::initialize: KBE_RES_PATH=C:/Users/hyc/Desktop/kbengine/kbe/res/;C:/Users/hyc/Desktop/kbengine/llgame/;C:/Users/hyc/Desktop/kbengine/llgame/scripts/;C:/Users/hyc/Desktop/kbengine/llgame/res/
  INFO root [2017-11-06 14:24:10,157] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/hyc/Desktop/kbengine/kbe/bin/server/
  INFO root [2017-11-06 14:24:10,158] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 172.16.4.154 ( 172.16.4.154:20013 )
  INFO root [2017-11-06 14:24:10,159] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 172.16.4.154:20013, SOMAXCONN=128.
  INFO root [2017-11-06 14:24:10,160] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 172.16.4.154 ( 172.16.4.154:61279 )
  INFO root [2017-11-06 14:24:10,161] - NetworkInterface::recreateListeningSocket(INTERNAL): address 172.16.4.154:61279, SOMAXCONN=128.
  INFO root [2017-11-06 14:24:10,161] - server-configs:
  INFO root [2017-11-06 14:24:10,161] - 	internalAddr : 172.16.4.154:61279
  INFO root [2017-11-06 14:24:10,162] - 	externalAddr : 172.16.4.154:20013
  INFO root [2017-11-06 14:24:10,162] - 	componentID : 9000
  INFO root [2017-11-06 14:24:10,601] - Components::findLogger: found logger, addr:172.16.4.154:61261
  INFO root [2017-11-06 14:24:10,601] - Components::addComponent: reset orderLog, uid=1001!
  INFO root [2017-11-06 14:24:10,601] - Components::addComponent[logger], uid=1001, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2017-11-06 14:24:12,305] - ---- loginapp Version: 0.9.12. ScriptVersion: 0.1.0. Protocol: 75C8DA776708D1E89693088467E5A4FB. Config: Debug X86. Built: 16:13:51 Jul  7 2017. AppID: 9000. UID: 1001. PID: 76796 ----
  INFO root [2017-11-06 14:24:12,306] - Server : Intel(R) Core(TM) i5-4590 CPU @ 3.30GHz with 4194303kB RAM
  INFO root [2017-11-06 14:24:12,306] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2017-11-06 14:24:12,306] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2017-11-06 14:24:12,466] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2017-11-06 14:24:12,466] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2017-11-06 14:24:12,467] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2017-11-06 14:24:12,469] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2017-11-06 14:24:12,469] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2017-11-06 14:24:12,469] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2017-11-06 14:24:12,503] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2017-11-06 14:24:12,534] - Script::install(): is successfully, Python=(3.4.2 (default, Jul  7 2017, 16:13:50) [MSC v.1800 32 bit (Intel)])!
  INFO root [2017-11-06 14:24:12,534] - Script::install(): KBExtra is successfully!
S_INFO root [2017-11-06 14:24:12,537] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2017-11-06 14:24:12,537] - TelnetServer server is running on port 31000
  INFO root [2017-11-06 14:24:12,751] - ---- loginapp is running ----
 DEBUG root [2017-11-06 14:24:12,751] - Components::process(): Request for the process of identity...
  WARN root [2017-11-06 14:24:18,904] - Forwarding logs to logger[172.16.4.154:61261/0/0/0]...
 ERROR root [2017-11-06 14:30:02,122] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2017-11-06 14:30:02,124] - The following logs sent to logger failed:
  WARN root [2017-11-06 14:30:02,124] - ==> [2017-11-06 14:30:02 122] TCPPacketReceiver::processPendingEvents(172.16.4.154:61261): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2017-11-06 14:30:02,124] - ==> [2017-11-06 14:30:02 122] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [2017-11-06 14:30:02,334] - TCPPacketReceiver::processPendingEvents(172.16.4.154:61262): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2017-11-06 14:30:02,335] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [2017-11-06 14:30:02,344] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2017-11-06 14:30:02,346] - onLoginAppShutDown()
  INFO root [2017-11-06 14:30:02,346] - Shutdowner::onShutdown: shutting down
  INFO root [2017-11-06 14:30:02,346] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2017-11-06 14:30:02,347] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [2017-11-06 14:30:02,347] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [2017-11-06 14:30:02,347] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [2017-11-06 14:30:02,347] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [2017-11-06 14:30:02,347] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [2017-11-06 14:30:02,347] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [2017-11-06 14:30:02,347] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [2017-11-06 14:30:02,356] - Script::uninstall(): is successfully!
 DEBUG root [2017-11-06 14:30:02,357] - ThreadPool::destroy(): starting size 2.
  WARN root [2017-11-06 14:30:02,657] - ThreadPool::destroy(): waiting for thread(2)[0x1d556a8,0x1d55778,], try=1
 DEBUG root [2017-11-06 14:30:03,058] - TPThread::~TPThread(): 0x1d556a8
 DEBUG root [2017-11-06 14:30:03,058] - TPThread::~TPThread(): 0x1d55778
 DEBUG root [2017-11-06 14:30:03,058] - ThreadPool::destroy(): successfully!
 DEBUG root [2017-11-06 14:30:03,067] - Bundle::destroyObjPool(): size 15.
 DEBUG root [2017-11-06 14:30:03,067] - Channel::destroyObjPool(): size 15.
 DEBUG root [2017-11-06 14:30:03,067] - TCPPacket::destroyObjPool(): size 15.
 DEBUG root [2017-11-06 14:30:03,067] - UDPPacket::destroyObjPool(): size 0.
 DEBUG root [2017-11-06 14:30:03,067] - EndPoint::destroyObjPool(): size 14.
 DEBUG root [2017-11-06 14:30:03,067] - Address::destroyObjPool(): size 0.
 DEBUG root [2017-11-06 14:30:03,067] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [2017-11-06 14:30:03,067] - UDPPacketReceiver::destroyObjPool(): size 0.
  INFO root [2017-11-06 14:30:03,067] - loginapp(9000) has shut down.
  INFO root [2017-11-06 14:30:07,771] - 
  INFO root [2017-11-06 14:30:07,772] - -----------------------------------------------------------------------------------------


  INFO root [2017-11-06 14:30:07,772] - Resmgr::initialize: KBE_ROOT=C:/Users/hyc/Desktop/kbengine/
  INFO root [2017-11-06 14:30:07,773] - Resmgr::initialize: KBE_RES_PATH=C:/Users/hyc/Desktop/kbengine/kbe/res/;C:/Users/hyc/Desktop/kbengine/llgame/;C:/Users/hyc/Desktop/kbengine/llgame/scripts/;C:/Users/hyc/Desktop/kbengine/llgame/res/
  INFO root [2017-11-06 14:30:07,773] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/hyc/Desktop/kbengine/kbe/bin/server/
  INFO root [2017-11-06 14:30:07,774] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 172.16.4.154 ( 172.16.4.154:20013 )
  INFO root [2017-11-06 14:30:07,775] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 172.16.4.154:20013, SOMAXCONN=128.
  INFO root [2017-11-06 14:30:07,776] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 172.16.4.154 ( 172.16.4.154:61336 )
  INFO root [2017-11-06 14:30:07,777] - NetworkInterface::recreateListeningSocket(INTERNAL): address 172.16.4.154:61336, SOMAXCONN=128.
  INFO root [2017-11-06 14:30:07,777] - server-configs:
  INFO root [2017-11-06 14:30:07,777] - 	internalAddr : 172.16.4.154:61336
  INFO root [2017-11-06 14:30:07,778] - 	externalAddr : 172.16.4.154:20013
  INFO root [2017-11-06 14:30:07,778] - 	componentID : 9000
  INFO root [2017-11-06 14:30:08,118] - Components::findLogger: found logger, addr:172.16.4.154:61331
  INFO root [2017-11-06 14:30:08,118] - Components::addComponent: reset orderLog, uid=1001!
  INFO root [2017-11-06 14:30:08,119] - Components::addComponent[logger], uid=1001, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2017-11-06 14:30:08,123] - ---- loginapp Version: 0.9.12. ScriptVersion: 0.1.0. Protocol: 75C8DA776708D1E89693088467E5A4FB. Config: Debug X86. Built: 16:13:51 Jul  7 2017. AppID: 9000. UID: 1001. PID: 88796 ----
  INFO root [2017-11-06 14:30:08,123] - Server : Intel(R) Core(TM) i5-4590 CPU @ 3.30GHz with 4194303kB RAM
  INFO root [2017-11-06 14:30:08,123] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2017-11-06 14:30:08,124] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2017-11-06 14:30:08,253] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2017-11-06 14:30:08,253] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2017-11-06 14:30:08,253] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2017-11-06 14:30:08,255] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2017-11-06 14:30:08,255] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2017-11-06 14:30:08,256] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2017-11-06 14:30:08,292] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2017-11-06 14:30:08,324] - Script::install(): is successfully, Python=(3.4.2 (default, Jul  7 2017, 16:13:50) [MSC v.1800 32 bit (Intel)])!
  INFO root [2017-11-06 14:30:08,324] - Script::install(): KBExtra is successfully!
S_INFO root [2017-11-06 14:30:08,327] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2017-11-06 14:30:08,327] - TelnetServer server is running on port 31000
  INFO root [2017-11-06 14:30:08,540] - ---- loginapp is running ----
 DEBUG root [2017-11-06 14:30:08,540] - Components::process(): Request for the process of identity...
  WARN root [2017-11-06 14:30:14,690] - Forwarding logs to logger[172.16.4.154:61331/0/0/0]...
 ERROR root [2017-11-06 14:30:28,806] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2017-11-06 14:30:28,824] - The following logs sent to logger failed:
  WARN root [2017-11-06 14:30:28,824] - ==> [2017-11-06 14:30:28 806] TCPPacketReceiver::processPendingEvents(172.16.4.154:61331): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2017-11-06 14:30:28,824] - ==> [2017-11-06 14:30:28 806] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [2017-11-06 14:30:29,020] - TCPPacketReceiver::processPendingEvents(172.16.4.154:61329): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2017-11-06 14:30:29,024] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [2017-11-06 14:30:29,025] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2017-11-06 14:30:29,025] - onLoginAppShutDown()
  INFO root [2017-11-06 14:30:29,025] - Shutdowner::onShutdown: shutting down
  INFO root [2017-11-06 14:30:29,025] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2017-11-06 14:30:29,026] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [2017-11-06 14:30:29,026] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [2017-11-06 14:30:29,026] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [2017-11-06 14:30:29,026] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [2017-11-06 14:30:29,026] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [2017-11-06 14:30:29,026] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [2017-11-06 14:30:29,026] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [2017-11-06 14:30:29,039] - Script::uninstall(): is successfully!
 DEBUG root [2017-11-06 14:30:29,039] - ThreadPool::destroy(): starting size 2.
  WARN root [2017-11-06 14:30:29,340] - ThreadPool::destroy(): waiting for thread(2)[0xb15b20,0xb15b88,], try=1
 DEBUG root [2017-11-06 14:30:29,742] - TPThread::~TPThread(): 0xb15b20
 DEBUG root [2017-11-06 14:30:29,742] - TPThread::~TPThread(): 0xb15b88
 DEBUG root [2017-11-06 14:30:29,742] - ThreadPool::destroy(): successfully!
 DEBUG root [2017-11-06 14:30:29,750] - Bundle::destroyObjPool(): size 13.
 DEBUG root [2017-11-06 14:30:29,750] - Channel::destroyObjPool(): size 15.
 DEBUG root [2017-11-06 14:30:29,750] - TCPPacket::destroyObjPool(): size 12.
 DEBUG root [2017-11-06 14:30:29,751] - UDPPacket::destroyObjPool(): size 2.
 DEBUG root [2017-11-06 14:30:29,751] - EndPoint::destroyObjPool(): size 14.
 DEBUG root [2017-11-06 14:30:29,751] - Address::destroyObjPool(): size 0.
 DEBUG root [2017-11-06 14:30:29,751] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [2017-11-06 14:30:29,751] - UDPPacketReceiver::destroyObjPool(): size 0.
  INFO root [2017-11-06 14:30:29,751] - loginapp(9000) has shut down.
  INFO root [2017-11-06 14:31:12,104] - 
  INFO root [2017-11-06 14:31:12,104] - -----------------------------------------------------------------------------------------


  INFO root [2017-11-06 14:31:12,105] - Resmgr::initialize: KBE_ROOT=C:/Users/hyc/Desktop/kbengine/
  INFO root [2017-11-06 14:31:12,105] - Resmgr::initialize: KBE_RES_PATH=C:/Users/hyc/Desktop/kbengine/kbe/res/;C:/Users/hyc/Desktop/kbengine/llgame/;C:/Users/hyc/Desktop/kbengine/llgame/scripts/;C:/Users/hyc/Desktop/kbengine/llgame/res/
  INFO root [2017-11-06 14:31:12,105] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/hyc/Desktop/kbengine/kbe/bin/server/
  INFO root [2017-11-06 14:31:12,107] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 172.16.4.154 ( 172.16.4.154:20013 )
  INFO root [2017-11-06 14:31:12,107] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 172.16.4.154:20013, SOMAXCONN=128.
  INFO root [2017-11-06 14:31:12,108] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 172.16.4.154 ( 172.16.4.154:61383 )
  INFO root [2017-11-06 14:31:12,109] - NetworkInterface::recreateListeningSocket(INTERNAL): address 172.16.4.154:61383, SOMAXCONN=128.
  INFO root [2017-11-06 14:31:12,109] - server-configs:
  INFO root [2017-11-06 14:31:12,109] - 	internalAddr : 172.16.4.154:61383
  INFO root [2017-11-06 14:31:12,109] - 	externalAddr : 172.16.4.154:20013
  INFO root [2017-11-06 14:31:12,109] - 	componentID : 9000
  INFO root [2017-11-06 14:31:12,498] - Components::findLogger: found logger, addr:172.16.4.154:61377
  INFO root [2017-11-06 14:31:12,498] - Components::addComponent: reset orderLog, uid=1001!
  INFO root [2017-11-06 14:31:12,498] - Components::addComponent[logger], uid=1001, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2017-11-06 14:31:12,503] - ---- loginapp Version: 0.9.12. ScriptVersion: 0.1.0. Protocol: 75C8DA776708D1E89693088467E5A4FB. Config: Debug X86. Built: 16:13:51 Jul  7 2017. AppID: 9000. UID: 1001. PID: 88396 ----
  INFO root [2017-11-06 14:31:12,503] - Server : Intel(R) Core(TM) i5-4590 CPU @ 3.30GHz with 4194303kB RAM
  INFO root [2017-11-06 14:31:12,504] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2017-11-06 14:31:12,504] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2017-11-06 14:31:12,632] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2017-11-06 14:31:12,632] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2017-11-06 14:31:12,632] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2017-11-06 14:31:12,634] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2017-11-06 14:31:12,634] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2017-11-06 14:31:12,634] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2017-11-06 14:31:12,667] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2017-11-06 14:31:12,698] - Script::install(): is successfully, Python=(3.4.2 (default, Jul  7 2017, 16:13:50) [MSC v.1800 32 bit (Intel)])!
  INFO root [2017-11-06 14:31:12,698] - Script::install(): KBExtra is successfully!
S_INFO root [2017-11-06 14:31:12,700] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2017-11-06 14:31:12,700] - TelnetServer server is running on port 31000
  INFO root [2017-11-06 14:31:12,917] - ---- loginapp is running ----
 DEBUG root [2017-11-06 14:31:12,918] - Components::process(): Request for the process of identity...
  WARN root [2017-11-06 14:31:19,112] - Forwarding logs to logger[172.16.4.154:61377/0/0/0]...
 ERROR root [2017-11-06 14:32:13,745] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2017-11-06 14:32:13,750] - The following logs sent to logger failed:
  WARN root [2017-11-06 14:32:13,751] - ==> [2017-11-06 14:32:13 745] TCPPacketReceiver::processPendingEvents(172.16.4.154:61377): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2017-11-06 14:32:13,751] - ==> [2017-11-06 14:32:13 745] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [2017-11-06 14:32:13,957] - TCPPacketReceiver::processPendingEvents(172.16.4.154:61378): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2017-11-06 14:32:13,958] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [2017-11-06 14:32:13,963] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2017-11-06 14:32:13,964] - onLoginAppShutDown()
  INFO root [2017-11-06 14:32:13,966] - Shutdowner::onShutdown: shutting down
  INFO root [2017-11-06 14:32:13,966] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2017-11-06 14:32:13,967] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [2017-11-06 14:32:13,967] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [2017-11-06 14:32:13,967] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [2017-11-06 14:32:13,967] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [2017-11-06 14:32:13,967] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [2017-11-06 14:32:13,967] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [2017-11-06 14:32:13,967] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [2017-11-06 14:32:13,980] - Script::uninstall(): is successfully!
 DEBUG root [2017-11-06 14:32:13,980] - ThreadPool::destroy(): starting size 2.
  WARN root [2017-11-06 14:32:14,281] - ThreadPool::destroy(): waiting for thread(2)[0x20f5090,0x20f50f8,], try=1
 DEBUG root [2017-11-06 14:32:14,683] - TPThread::~TPThread(): 0x20f5090
 DEBUG root [2017-11-06 14:32:14,683] - TPThread::~TPThread(): 0x20f50f8
 DEBUG root [2017-11-06 14:32:14,683] - ThreadPool::destroy(): successfully!
 DEBUG root [2017-11-06 14:32:14,691] - Bundle::destroyObjPool(): size 13.
 DEBUG root [2017-11-06 14:32:14,691] - Channel::destroyObjPool(): size 15.
 DEBUG root [2017-11-06 14:32:14,691] - TCPPacket::destroyObjPool(): size 12.
 DEBUG root [2017-11-06 14:32:14,691] - UDPPacket::destroyObjPool(): size 15.
 DEBUG root [2017-11-06 14:32:14,691] - EndPoint::destroyObjPool(): size 14.
 DEBUG root [2017-11-06 14:32:14,691] - Address::destroyObjPool(): size 0.
 DEBUG root [2017-11-06 14:32:14,691] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [2017-11-06 14:32:14,691] - UDPPacketReceiver::destroyObjPool(): size 0.
  INFO root [2017-11-06 14:32:14,691] - loginapp(9000) has shut down.
  INFO root [2017-11-06 14:32:19,858] - 
  INFO root [2017-11-06 14:32:19,858] - -----------------------------------------------------------------------------------------


  INFO root [2017-11-06 14:32:19,859] - Resmgr::initialize: KBE_ROOT=C:/Users/hyc/Desktop/kbengine/
  INFO root [2017-11-06 14:32:19,859] - Resmgr::initialize: KBE_RES_PATH=C:/Users/hyc/Desktop/kbengine/kbe/res/;C:/Users/hyc/Desktop/kbengine/llgame/;C:/Users/hyc/Desktop/kbengine/llgame/scripts/;C:/Users/hyc/Desktop/kbengine/llgame/res/
  INFO root [2017-11-06 14:32:19,859] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/hyc/Desktop/kbengine/kbe/bin/server/
  INFO root [2017-11-06 14:32:19,861] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 172.16.4.154 ( 172.16.4.154:20013 )
  INFO root [2017-11-06 14:32:19,862] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 172.16.4.154:20013, SOMAXCONN=128.
  INFO root [2017-11-06 14:32:19,862] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 172.16.4.154 ( 172.16.4.154:61431 )
  INFO root [2017-11-06 14:32:19,863] - NetworkInterface::recreateListeningSocket(INTERNAL): address 172.16.4.154:61431, SOMAXCONN=128.
  INFO root [2017-11-06 14:32:19,864] - server-configs:
  INFO root [2017-11-06 14:32:19,864] - 	internalAddr : 172.16.4.154:61431
  INFO root [2017-11-06 14:32:19,864] - 	externalAddr : 172.16.4.154:20013
  INFO root [2017-11-06 14:32:19,864] - 	componentID : 9000
  INFO root [2017-11-06 14:32:20,321] - Components::findLogger: found logger, addr:172.16.4.154:61426
  INFO root [2017-11-06 14:32:20,321] - Components::addComponent: reset orderLog, uid=1001!
  INFO root [2017-11-06 14:32:20,321] - Components::addComponent[logger], uid=1001, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2017-11-06 14:32:20,326] - ---- loginapp Version: 0.9.12. ScriptVersion: 0.1.0. Protocol: 75C8DA776708D1E89693088467E5A4FB. Config: Debug X86. Built: 16:13:51 Jul  7 2017. AppID: 9000. UID: 1001. PID: 89972 ----
  INFO root [2017-11-06 14:32:20,326] - Server : Intel(R) Core(TM) i5-4590 CPU @ 3.30GHz with 4194303kB RAM
  INFO root [2017-11-06 14:32:20,326] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2017-11-06 14:32:20,326] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2017-11-06 14:32:20,452] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2017-11-06 14:32:20,453] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2017-11-06 14:32:20,453] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2017-11-06 14:32:20,454] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2017-11-06 14:32:20,454] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2017-11-06 14:32:20,455] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2017-11-06 14:32:20,485] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2017-11-06 14:32:20,514] - Script::install(): is successfully, Python=(3.4.2 (default, Jul  7 2017, 16:13:50) [MSC v.1800 32 bit (Intel)])!
  INFO root [2017-11-06 14:32:20,514] - Script::install(): KBExtra is successfully!
S_INFO root [2017-11-06 14:32:20,516] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2017-11-06 14:32:20,517] - TelnetServer server is running on port 31000
  INFO root [2017-11-06 14:32:20,728] - ---- loginapp is running ----
 DEBUG root [2017-11-06 14:32:20,728] - Components::process(): Request for the process of identity...
  WARN root [2017-11-06 14:32:26,861] - Forwarding logs to logger[172.16.4.154:61426/0/0/0]...
 ERROR root [2017-11-06 16:38:42,159] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2017-11-06 16:38:42,174] - The following logs sent to logger failed:
  WARN root [2017-11-06 16:38:42,174] - ==> [2017-11-06 16:38:42 159] TCPPacketReceiver::processPendingEvents(172.16.4.154:61426): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2017-11-06 16:38:42,175] - ==> [2017-11-06 16:38:42 159] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [2017-11-06 16:38:42,347] - TCPPacketReceiver::processPendingEvents(172.16.4.154:61428): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2017-11-06 16:38:42,351] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [2017-11-06 16:38:42,352] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2017-11-06 16:38:42,366] - onLoginAppShutDown()
  INFO root [2017-11-06 16:38:42,366] - Shutdowner::onShutdown: shutting down
  INFO root [2017-11-06 16:38:42,366] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2017-11-06 16:38:42,366] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [2017-11-06 16:38:42,366] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [2017-11-06 16:38:42,367] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [2017-11-06 16:38:42,367] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [2017-11-06 16:38:42,367] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [2017-11-06 16:38:42,367] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [2017-11-06 16:38:42,367] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [2017-11-06 16:38:42,375] - Script::uninstall(): is successfully!
 DEBUG root [2017-11-06 16:38:42,376] - ThreadPool::destroy(): starting size 2.
  WARN root [2017-11-06 16:38:42,676] - ThreadPool::destroy(): waiting for thread(2)[0x615918,0x615bf0,], try=1
 DEBUG root [2017-11-06 16:38:43,077] - TPThread::~TPThread(): 0x615918
 DEBUG root [2017-11-06 16:38:43,077] - TPThread::~TPThread(): 0x615bf0
 DEBUG root [2017-11-06 16:38:43,077] - ThreadPool::destroy(): successfully!
 DEBUG root [2017-11-06 16:38:43,086] - Bundle::destroyObjPool(): size 13.
 DEBUG root [2017-11-06 16:38:43,086] - Channel::destroyObjPool(): size 15.
 DEBUG root [2017-11-06 16:38:43,086] - TCPPacket::destroyObjPool(): size 1.
 DEBUG root [2017-11-06 16:38:43,086] - UDPPacket::destroyObjPool(): size 1.
 DEBUG root [2017-11-06 16:38:43,086] - EndPoint::destroyObjPool(): size 14.
 DEBUG root [2017-11-06 16:38:43,086] - Address::destroyObjPool(): size 0.
 DEBUG root [2017-11-06 16:38:43,086] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [2017-11-06 16:38:43,086] - UDPPacketReceiver::destroyObjPool(): size 0.
  INFO root [2017-11-06 16:38:43,086] - loginapp(9000) has shut down.
  INFO root [2017-11-06 16:38:57,338] - 
  INFO root [2017-11-06 16:38:57,338] - -----------------------------------------------------------------------------------------


  INFO root [2017-11-06 16:38:57,339] - Resmgr::initialize: KBE_ROOT=C:/Users/hyc/Desktop/kbengine/
  INFO root [2017-11-06 16:38:57,339] - Resmgr::initialize: KBE_RES_PATH=C:/Users/hyc/Desktop/kbengine/kbe/res/;C:/Users/hyc/Desktop/kbengine/llgame/;C:/Users/hyc/Desktop/kbengine/llgame/scripts/;C:/Users/hyc/Desktop/kbengine/llgame/res/
  INFO root [2017-11-06 16:38:57,339] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/hyc/Desktop/kbengine/kbe/bin/server/
  INFO root [2017-11-06 16:38:57,340] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 172.16.4.154 ( 172.16.4.154:20013 )
  INFO root [2017-11-06 16:38:57,341] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 172.16.4.154:20013, SOMAXCONN=128.
  INFO root [2017-11-06 16:38:57,342] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 172.16.4.154 ( 172.16.4.154:62024 )
  INFO root [2017-11-06 16:38:57,343] - NetworkInterface::recreateListeningSocket(INTERNAL): address 172.16.4.154:62024, SOMAXCONN=128.
  INFO root [2017-11-06 16:38:57,343] - server-configs:
  INFO root [2017-11-06 16:38:57,343] - 	internalAddr : 172.16.4.154:62024
  INFO root [2017-11-06 16:38:57,343] - 	externalAddr : 172.16.4.154:20013
  INFO root [2017-11-06 16:38:57,343] - 	componentID : 9000
  INFO root [2017-11-06 16:38:57,727] - Components::findLogger: found logger, addr:172.16.4.154:62018
  INFO root [2017-11-06 16:38:57,727] - Components::addComponent: reset orderLog, uid=1001!
  INFO root [2017-11-06 16:38:57,728] - Components::addComponent[logger], uid=1001, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2017-11-06 16:38:57,732] - ---- loginapp Version: 0.9.12. ScriptVersion: 0.1.0. Protocol: 75C8DA776708D1E89693088467E5A4FB. Config: Debug X86. Built: 16:13:51 Jul  7 2017. AppID: 9000. UID: 1001. PID: 90044 ----
  INFO root [2017-11-06 16:38:57,732] - Server : Intel(R) Core(TM) i5-4590 CPU @ 3.30GHz with 4194303kB RAM
  INFO root [2017-11-06 16:38:57,732] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2017-11-06 16:38:57,732] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2017-11-06 16:38:57,859] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2017-11-06 16:38:57,860] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2017-11-06 16:38:57,860] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2017-11-06 16:38:57,862] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2017-11-06 16:38:57,862] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2017-11-06 16:38:57,862] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2017-11-06 16:38:57,897] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2017-11-06 16:38:57,928] - Script::install(): is successfully, Python=(3.4.2 (default, Jul  7 2017, 16:13:50) [MSC v.1800 32 bit (Intel)])!
  INFO root [2017-11-06 16:38:57,928] - Script::install(): KBExtra is successfully!
S_INFO root [2017-11-06 16:38:57,930] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2017-11-06 16:38:57,930] - TelnetServer server is running on port 31000
  INFO root [2017-11-06 16:38:58,143] - ---- loginapp is running ----
 DEBUG root [2017-11-06 16:38:58,143] - Components::process(): Request for the process of identity...
  WARN root [2017-11-06 16:39:04,268] - Forwarding logs to logger[172.16.4.154:62018/0/0/0]...
 ERROR root [2017-11-06 16:44:45,254] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2017-11-06 16:44:45,256] - The following logs sent to logger failed:
  WARN root [2017-11-06 16:44:45,256] - ==> [2017-11-06 16:44:45 254] TCPPacketReceiver::processPendingEvents(172.16.4.154:62018): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2017-11-06 16:44:45,256] - ==> [2017-11-06 16:44:45 254] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [2017-11-06 16:44:45,475] - TCPPacketReceiver::processPendingEvents(172.16.4.154:62019): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2017-11-06 16:44:45,476] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [2017-11-06 16:44:45,482] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2017-11-06 16:44:45,487] - onLoginAppShutDown()
  INFO root [2017-11-06 16:44:45,487] - Shutdowner::onShutdown: shutting down
  INFO root [2017-11-06 16:44:45,487] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2017-11-06 16:44:45,489] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [2017-11-06 16:44:45,490] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [2017-11-06 16:44:45,490] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [2017-11-06 16:44:45,491] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [2017-11-06 16:44:45,491] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [2017-11-06 16:44:45,491] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [2017-11-06 16:44:45,491] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [2017-11-06 16:44:45,502] - Script::uninstall(): is successfully!
 DEBUG root [2017-11-06 16:44:45,502] - ThreadPool::destroy(): starting size 2.
  WARN root [2017-11-06 16:44:45,803] - ThreadPool::destroy(): waiting for thread(2)[0x2055c58,0x2055848,], try=1
 DEBUG root [2017-11-06 16:44:46,205] - TPThread::~TPThread(): 0x2055c58
 DEBUG root [2017-11-06 16:44:46,205] - TPThread::~TPThread(): 0x2055848
 DEBUG root [2017-11-06 16:44:46,205] - ThreadPool::destroy(): successfully!
 DEBUG root [2017-11-06 16:44:46,213] - Bundle::destroyObjPool(): size 15.
 DEBUG root [2017-11-06 16:44:46,213] - Channel::destroyObjPool(): size 15.
 DEBUG root [2017-11-06 16:44:46,213] - TCPPacket::destroyObjPool(): size 15.
 DEBUG root [2017-11-06 16:44:46,214] - UDPPacket::destroyObjPool(): size 2.
 DEBUG root [2017-11-06 16:44:46,214] - EndPoint::destroyObjPool(): size 14.
 DEBUG root [2017-11-06 16:44:46,214] - Address::destroyObjPool(): size 0.
 DEBUG root [2017-11-06 16:44:46,214] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [2017-11-06 16:44:46,214] - UDPPacketReceiver::destroyObjPool(): size 0.
  INFO root [2017-11-06 16:44:46,214] - loginapp(9000) has shut down.
  INFO root [2017-11-06 16:44:51,482] - 
  INFO root [2017-11-06 16:44:51,482] - -----------------------------------------------------------------------------------------


  INFO root [2017-11-06 16:44:51,483] - Resmgr::initialize: KBE_ROOT=C:/Users/hyc/Desktop/kbengine/
  INFO root [2017-11-06 16:44:51,483] - Resmgr::initialize: KBE_RES_PATH=C:/Users/hyc/Desktop/kbengine/kbe/res/;C:/Users/hyc/Desktop/kbengine/llgame/;C:/Users/hyc/Desktop/kbengine/llgame/scripts/;C:/Users/hyc/Desktop/kbengine/llgame/res/
  INFO root [2017-11-06 16:44:51,483] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/hyc/Desktop/kbengine/kbe/bin/server/
  INFO root [2017-11-06 16:44:51,484] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 172.16.4.154 ( 172.16.4.154:20013 )
  INFO root [2017-11-06 16:44:51,485] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 172.16.4.154:20013, SOMAXCONN=128.
  INFO root [2017-11-06 16:44:51,486] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 172.16.4.154 ( 172.16.4.154:62082 )
  INFO root [2017-11-06 16:44:51,487] - NetworkInterface::recreateListeningSocket(INTERNAL): address 172.16.4.154:62082, SOMAXCONN=128.
  INFO root [2017-11-06 16:44:51,487] - server-configs:
  INFO root [2017-11-06 16:44:51,487] - 	internalAddr : 172.16.4.154:62082
  INFO root [2017-11-06 16:44:51,487] - 	externalAddr : 172.16.4.154:20013
  INFO root [2017-11-06 16:44:51,487] - 	componentID : 9000
  INFO root [2017-11-06 16:44:51,845] - Components::findLogger: found logger, addr:172.16.4.154:62076
  INFO root [2017-11-06 16:44:51,845] - Components::addComponent: reset orderLog, uid=1001!
  INFO root [2017-11-06 16:44:51,845] - Components::addComponent[logger], uid=1001, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2017-11-06 16:44:51,849] - ---- loginapp Version: 0.9.12. ScriptVersion: 0.1.0. Protocol: 75C8DA776708D1E89693088467E5A4FB. Config: Debug X86. Built: 16:13:51 Jul  7 2017. AppID: 9000. UID: 1001. PID: 88092 ----
  INFO root [2017-11-06 16:44:51,849] - Server : Intel(R) Core(TM) i5-4590 CPU @ 3.30GHz with 4194303kB RAM
  INFO root [2017-11-06 16:44:51,849] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2017-11-06 16:44:51,850] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2017-11-06 16:44:51,977] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2017-11-06 16:44:51,978] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2017-11-06 16:44:51,978] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2017-11-06 16:44:51,980] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2017-11-06 16:44:51,980] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2017-11-06 16:44:51,980] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2017-11-06 16:44:52,013] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2017-11-06 16:44:52,045] - Script::install(): is successfully, Python=(3.4.2 (default, Jul  7 2017, 16:13:50) [MSC v.1800 32 bit (Intel)])!
  INFO root [2017-11-06 16:44:52,045] - Script::install(): KBExtra is successfully!
S_INFO root [2017-11-06 16:44:52,047] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2017-11-06 16:44:52,048] - TelnetServer server is running on port 31000
  INFO root [2017-11-06 16:44:52,258] - ---- loginapp is running ----
 DEBUG root [2017-11-06 16:44:52,258] - Components::process(): Request for the process of identity...
  WARN root [2017-11-06 16:44:58,407] - Forwarding logs to logger[172.16.4.154:62076/0/0/0]...
 ERROR root [2017-11-06 17:02:14,069] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2017-11-06 17:02:14,073] - The following logs sent to logger failed:
  WARN root [2017-11-06 17:02:14,073] - ==> [2017-11-06 17:02:14 069] TCPPacketReceiver::processPendingEvents(172.16.4.154:62076): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2017-11-06 17:02:14,073] - ==> [2017-11-06 17:02:14 070] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [2017-11-06 17:02:14,308] - TCPPacketReceiver::processPendingEvents(172.16.4.154:62077): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2017-11-06 17:02:14,311] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [2017-11-06 17:02:14,312] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2017-11-06 17:02:14,312] - onLoginAppShutDown()
  INFO root [2017-11-06 17:02:14,312] - Shutdowner::onShutdown: shutting down
  INFO root [2017-11-06 17:02:14,313] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2017-11-06 17:02:14,313] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [2017-11-06 17:02:14,313] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [2017-11-06 17:02:14,313] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [2017-11-06 17:02:14,313] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [2017-11-06 17:02:14,313] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [2017-11-06 17:02:14,313] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [2017-11-06 17:02:14,313] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [2017-11-06 17:02:14,323] - Script::uninstall(): is successfully!
 DEBUG root [2017-11-06 17:02:14,323] - ThreadPool::destroy(): starting size 2.
  WARN root [2017-11-06 17:02:14,624] - ThreadPool::destroy(): waiting for thread(2)[0xba5b88,0xba5918,], try=1
 DEBUG root [2017-11-06 17:02:15,027] - TPThread::~TPThread(): 0xba5b88
 DEBUG root [2017-11-06 17:02:15,027] - TPThread::~TPThread(): 0xba5918
 DEBUG root [2017-11-06 17:02:15,027] - ThreadPool::destroy(): successfully!
 DEBUG root [2017-11-06 17:02:15,035] - Bundle::destroyObjPool(): size 13.
 DEBUG root [2017-11-06 17:02:15,035] - Channel::destroyObjPool(): size 15.
 DEBUG root [2017-11-06 17:02:15,035] - TCPPacket::destroyObjPool(): size 1.
 DEBUG root [2017-11-06 17:02:15,035] - UDPPacket::destroyObjPool(): size 2.
 DEBUG root [2017-11-06 17:02:15,036] - EndPoint::destroyObjPool(): size 14.
 DEBUG root [2017-11-06 17:02:15,036] - Address::destroyObjPool(): size 0.
 DEBUG root [2017-11-06 17:02:15,036] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [2017-11-06 17:02:15,036] - UDPPacketReceiver::destroyObjPool(): size 0.
  INFO root [2017-11-06 17:02:15,036] - loginapp(9000) has shut down.
  INFO root [2017-11-06 17:02:20,479] - 
  INFO root [2017-11-06 17:02:20,479] - -----------------------------------------------------------------------------------------


  INFO root [2017-11-06 17:02:20,480] - Resmgr::initialize: KBE_ROOT=C:/Users/hyc/Desktop/kbengine/
  INFO root [2017-11-06 17:02:20,480] - Resmgr::initialize: KBE_RES_PATH=C:/Users/hyc/Desktop/kbengine/kbe/res/;C:/Users/hyc/Desktop/kbengine/llgame/;C:/Users/hyc/Desktop/kbengine/llgame/scripts/;C:/Users/hyc/Desktop/kbengine/llgame/res/
  INFO root [2017-11-06 17:02:20,480] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/hyc/Desktop/kbengine/kbe/bin/server/
  INFO root [2017-11-06 17:02:20,481] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 172.16.4.154 ( 172.16.4.154:20013 )
  INFO root [2017-11-06 17:02:20,482] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 172.16.4.154:20013, SOMAXCONN=128.
  INFO root [2017-11-06 17:02:20,483] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 172.16.4.154 ( 172.16.4.154:51623 )
  INFO root [2017-11-06 17:02:20,484] - NetworkInterface::recreateListeningSocket(INTERNAL): address 172.16.4.154:51623, SOMAXCONN=128.
  INFO root [2017-11-06 17:02:20,484] - server-configs:
  INFO root [2017-11-06 17:02:20,484] - 	internalAddr : 172.16.4.154:51623
  INFO root [2017-11-06 17:02:20,484] - 	externalAddr : 172.16.4.154:20013
  INFO root [2017-11-06 17:02:20,484] - 	componentID : 9000
  INFO root [2017-11-06 17:02:20,925] - Components::findLogger: found logger, addr:172.16.4.154:51617
  INFO root [2017-11-06 17:02:20,925] - Components::addComponent: reset orderLog, uid=1001!
  INFO root [2017-11-06 17:02:20,925] - Components::addComponent[logger], uid=1001, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2017-11-06 17:02:20,930] - ---- loginapp Version: 0.9.12. ScriptVersion: 0.1.0. Protocol: 75C8DA776708D1E89693088467E5A4FB. Config: Debug X86. Built: 16:13:51 Jul  7 2017. AppID: 9000. UID: 1001. PID: 91236 ----
  INFO root [2017-11-06 17:02:20,930] - Server : Intel(R) Core(TM) i5-4590 CPU @ 3.30GHz with 4194303kB RAM
  INFO root [2017-11-06 17:02:20,930] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2017-11-06 17:02:20,930] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2017-11-06 17:02:21,058] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2017-11-06 17:02:21,058] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2017-11-06 17:02:21,058] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2017-11-06 17:02:21,060] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2017-11-06 17:02:21,060] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2017-11-06 17:02:21,061] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2017-11-06 17:02:21,092] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2017-11-06 17:02:21,124] - Script::install(): is successfully, Python=(3.4.2 (default, Jul  7 2017, 16:13:50) [MSC v.1800 32 bit (Intel)])!
  INFO root [2017-11-06 17:02:21,124] - Script::install(): KBExtra is successfully!
S_INFO root [2017-11-06 17:02:21,126] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2017-11-06 17:02:21,126] - TelnetServer server is running on port 31000
  INFO root [2017-11-06 17:02:21,339] - ---- loginapp is running ----
 DEBUG root [2017-11-06 17:02:21,339] - Components::process(): Request for the process of identity...
  WARN root [2017-11-06 17:02:27,484] - Forwarding logs to logger[172.16.4.154:51617/0/0/0]...
 ERROR root [2017-11-06 17:03:13,766] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2017-11-06 17:03:13,767] - The following logs sent to logger failed:
  WARN root [2017-11-06 17:03:13,767] - ==> [2017-11-06 17:03:13 762] TCPPacketReceiver::processPendingEvents(172.16.4.154:51617): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2017-11-06 17:03:13,768] - ==> [2017-11-06 17:03:13 767] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [2017-11-06 17:03:13,987] - TCPPacketReceiver::processPendingEvents(172.16.4.154:51619): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2017-11-06 17:03:13,989] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [2017-11-06 17:03:13,989] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2017-11-06 17:03:13,989] - onLoginAppShutDown()
  INFO root [2017-11-06 17:03:13,989] - Shutdowner::onShutdown: shutting down
  INFO root [2017-11-06 17:03:13,989] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2017-11-06 17:03:13,989] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [2017-11-06 17:03:13,989] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [2017-11-06 17:03:13,990] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [2017-11-06 17:03:13,990] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [2017-11-06 17:03:13,990] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [2017-11-06 17:03:13,990] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [2017-11-06 17:03:13,990] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [2017-11-06 17:03:13,998] - Script::uninstall(): is successfully!
 DEBUG root [2017-11-06 17:03:13,998] - ThreadPool::destroy(): starting size 2.
  WARN root [2017-11-06 17:03:14,299] - ThreadPool::destroy(): waiting for thread(2)[0xae54a0,0xae5298,], try=1
 DEBUG root [2017-11-06 17:03:14,701] - TPThread::~TPThread(): 0xae54a0
 DEBUG root [2017-11-06 17:03:14,701] - TPThread::~TPThread(): 0xae5298
 DEBUG root [2017-11-06 17:03:14,701] - ThreadPool::destroy(): successfully!
 DEBUG root [2017-11-06 17:03:14,709] - Bundle::destroyObjPool(): size 13.
 DEBUG root [2017-11-06 17:03:14,709] - Channel::destroyObjPool(): size 15.
 DEBUG root [2017-11-06 17:03:14,709] - TCPPacket::destroyObjPool(): size 12.
 DEBUG root [2017-11-06 17:03:14,709] - UDPPacket::destroyObjPool(): size 1.
 DEBUG root [2017-11-06 17:03:14,709] - EndPoint::destroyObjPool(): size 14.
 DEBUG root [2017-11-06 17:03:14,709] - Address::destroyObjPool(): size 0.
 DEBUG root [2017-11-06 17:03:14,709] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [2017-11-06 17:03:14,709] - UDPPacketReceiver::destroyObjPool(): size 0.
  INFO root [2017-11-06 17:03:14,709] - loginapp(9000) has shut down.
  INFO root [2017-11-06 17:03:39,337] - 
  INFO root [2017-11-06 17:03:39,337] - -----------------------------------------------------------------------------------------


  INFO root [2017-11-06 17:03:39,337] - Resmgr::initialize: KBE_ROOT=C:/Users/hyc/Desktop/kbengine/
  INFO root [2017-11-06 17:03:39,338] - Resmgr::initialize: KBE_RES_PATH=C:/Users/hyc/Desktop/kbengine/kbe/res/;C:/Users/hyc/Desktop/kbengine/llgame/;C:/Users/hyc/Desktop/kbengine/llgame/scripts/;C:/Users/hyc/Desktop/kbengine/llgame/res/
  INFO root [2017-11-06 17:03:39,338] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/hyc/Desktop/kbengine/kbe/bin/server/
  INFO root [2017-11-06 17:03:39,339] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 172.16.4.154 ( 172.16.4.154:20013 )
  INFO root [2017-11-06 17:03:39,340] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 172.16.4.154:20013, SOMAXCONN=128.
  INFO root [2017-11-06 17:03:39,341] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 172.16.4.154 ( 172.16.4.154:51671 )
  INFO root [2017-11-06 17:03:39,342] - NetworkInterface::recreateListeningSocket(INTERNAL): address 172.16.4.154:51671, SOMAXCONN=128.
  INFO root [2017-11-06 17:03:39,342] - server-configs:
  INFO root [2017-11-06 17:03:39,342] - 	internalAddr : 172.16.4.154:51671
  INFO root [2017-11-06 17:03:39,342] - 	externalAddr : 172.16.4.154:20013
  INFO root [2017-11-06 17:03:39,342] - 	componentID : 9000
  INFO root [2017-11-06 17:03:39,788] - Components::findLogger: found logger, addr:172.16.4.154:51665
  INFO root [2017-11-06 17:03:39,788] - Components::addComponent: reset orderLog, uid=1001!
  INFO root [2017-11-06 17:03:39,788] - Components::addComponent[logger], uid=1001, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2017-11-06 17:03:39,793] - ---- loginapp Version: 0.9.12. ScriptVersion: 0.1.0. Protocol: 75C8DA776708D1E89693088467E5A4FB. Config: Debug X86. Built: 16:13:51 Jul  7 2017. AppID: 9000. UID: 1001. PID: 91392 ----
  INFO root [2017-11-06 17:03:39,793] - Server : Intel(R) Core(TM) i5-4590 CPU @ 3.30GHz with 4194303kB RAM
  INFO root [2017-11-06 17:03:39,793] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2017-11-06 17:03:39,793] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2017-11-06 17:03:39,920] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2017-11-06 17:03:39,921] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2017-11-06 17:03:39,921] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2017-11-06 17:03:39,923] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2017-11-06 17:03:39,923] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2017-11-06 17:03:39,923] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2017-11-06 17:03:39,955] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2017-11-06 17:03:39,986] - Script::install(): is successfully, Python=(3.4.2 (default, Jul  7 2017, 16:13:50) [MSC v.1800 32 bit (Intel)])!
  INFO root [2017-11-06 17:03:39,986] - Script::install(): KBExtra is successfully!
S_INFO root [2017-11-06 17:03:39,988] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2017-11-06 17:03:39,989] - TelnetServer server is running on port 31000
  INFO root [2017-11-06 17:03:40,207] - ---- loginapp is running ----
 DEBUG root [2017-11-06 17:03:40,207] - Components::process(): Request for the process of identity...
  WARN root [2017-11-06 17:03:46,364] - Forwarding logs to logger[172.16.4.154:51665/0/0/0]...
 ERROR root [2017-11-06 17:05:40,470] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2017-11-06 17:05:40,474] - The following logs sent to logger failed:
  WARN root [2017-11-06 17:05:40,474] - ==> [2017-11-06 17:05:40 465] TCPPacketReceiver::processPendingEvents(172.16.4.154:51665): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2017-11-06 17:05:40,474] - ==> [2017-11-06 17:05:40 471] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [2017-11-06 17:05:40,694] - TCPPacketReceiver::processPendingEvents(172.16.4.154:51668): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2017-11-06 17:05:40,697] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [2017-11-06 17:05:40,699] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2017-11-06 17:05:40,705] - onLoginAppShutDown()
  INFO root [2017-11-06 17:05:40,705] - Shutdowner::onShutdown: shutting down
  INFO root [2017-11-06 17:05:40,705] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2017-11-06 17:05:40,705] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [2017-11-06 17:05:40,705] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [2017-11-06 17:05:40,705] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [2017-11-06 17:05:40,705] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [2017-11-06 17:05:40,705] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [2017-11-06 17:05:40,705] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [2017-11-06 17:05:40,706] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [2017-11-06 17:05:40,715] - Script::uninstall(): is successfully!
 DEBUG root [2017-11-06 17:05:40,715] - ThreadPool::destroy(): starting size 2.
  WARN root [2017-11-06 17:05:41,017] - ThreadPool::destroy(): waiting for thread(2)[0x1f95a50,0x1f95710,], try=1
 DEBUG root [2017-11-06 17:05:41,418] - TPThread::~TPThread(): 0x1f95a50
 DEBUG root [2017-11-06 17:05:41,418] - TPThread::~TPThread(): 0x1f95710
 DEBUG root [2017-11-06 17:05:41,418] - ThreadPool::destroy(): successfully!
 DEBUG root [2017-11-06 17:05:41,426] - Bundle::destroyObjPool(): size 13.
 DEBUG root [2017-11-06 17:05:41,426] - Channel::destroyObjPool(): size 15.
 DEBUG root [2017-11-06 17:05:41,426] - TCPPacket::destroyObjPool(): size 12.
 DEBUG root [2017-11-06 17:05:41,427] - UDPPacket::destroyObjPool(): size 2.
 DEBUG root [2017-11-06 17:05:41,427] - EndPoint::destroyObjPool(): size 14.
 DEBUG root [2017-11-06 17:05:41,427] - Address::destroyObjPool(): size 0.
 DEBUG root [2017-11-06 17:05:41,427] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [2017-11-06 17:05:41,427] - UDPPacketReceiver::destroyObjPool(): size 0.
  INFO root [2017-11-06 17:05:41,427] - loginapp(9000) has shut down.
  INFO root [2017-11-06 17:05:45,660] - 
  INFO root [2017-11-06 17:05:45,660] - -----------------------------------------------------------------------------------------


  INFO root [2017-11-06 17:05:45,661] - Resmgr::initialize: KBE_ROOT=C:/Users/hyc/Desktop/kbengine/
  INFO root [2017-11-06 17:05:45,661] - Resmgr::initialize: KBE_RES_PATH=C:/Users/hyc/Desktop/kbengine/kbe/res/;C:/Users/hyc/Desktop/kbengine/llgame/;C:/Users/hyc/Desktop/kbengine/llgame/scripts/;C:/Users/hyc/Desktop/kbengine/llgame/res/
  INFO root [2017-11-06 17:05:45,661] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/hyc/Desktop/kbengine/kbe/bin/server/
  INFO root [2017-11-06 17:05:45,663] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 172.16.4.154 ( 172.16.4.154:20013 )
  INFO root [2017-11-06 17:05:45,664] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 172.16.4.154:20013, SOMAXCONN=128.
  INFO root [2017-11-06 17:05:45,665] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 172.16.4.154 ( 172.16.4.154:51718 )
  INFO root [2017-11-06 17:05:45,666] - NetworkInterface::recreateListeningSocket(INTERNAL): address 172.16.4.154:51718, SOMAXCONN=128.
  INFO root [2017-11-06 17:05:45,666] - server-configs:
  INFO root [2017-11-06 17:05:45,666] - 	internalAddr : 172.16.4.154:51718
  INFO root [2017-11-06 17:05:45,666] - 	externalAddr : 172.16.4.154:20013
  INFO root [2017-11-06 17:05:45,666] - 	componentID : 9000
  INFO root [2017-11-06 17:05:46,034] - Components::findLogger: found logger, addr:172.16.4.154:51712
  INFO root [2017-11-06 17:05:46,034] - Components::addComponent: reset orderLog, uid=1001!
  INFO root [2017-11-06 17:05:46,034] - Components::addComponent[logger], uid=1001, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2017-11-06 17:05:46,038] - ---- loginapp Version: 0.9.12. ScriptVersion: 0.1.0. Protocol: 75C8DA776708D1E89693088467E5A4FB. Config: Debug X86. Built: 16:13:51 Jul  7 2017. AppID: 9000. UID: 1001. PID: 90456 ----
  INFO root [2017-11-06 17:05:46,038] - Server : Intel(R) Core(TM) i5-4590 CPU @ 3.30GHz with 4194303kB RAM
  INFO root [2017-11-06 17:05:46,039] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2017-11-06 17:05:46,039] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2017-11-06 17:05:46,165] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2017-11-06 17:05:46,166] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2017-11-06 17:05:46,166] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2017-11-06 17:05:46,168] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2017-11-06 17:05:46,168] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2017-11-06 17:05:46,168] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2017-11-06 17:05:46,199] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2017-11-06 17:05:46,230] - Script::install(): is successfully, Python=(3.4.2 (default, Jul  7 2017, 16:13:50) [MSC v.1800 32 bit (Intel)])!
  INFO root [2017-11-06 17:05:46,230] - Script::install(): KBExtra is successfully!
S_INFO root [2017-11-06 17:05:46,232] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2017-11-06 17:05:46,233] - TelnetServer server is running on port 31000
  INFO root [2017-11-06 17:05:46,444] - ---- loginapp is running ----
 DEBUG root [2017-11-06 17:05:46,444] - Components::process(): Request for the process of identity...
  WARN root [2017-11-06 17:05:52,592] - Forwarding logs to logger[172.16.4.154:51712/0/0/0]...
 ERROR root [2017-11-06 17:10:33,242] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2017-11-06 17:10:33,243] - The following logs sent to logger failed:
  WARN root [2017-11-06 17:10:33,244] - ==> [2017-11-06 17:10:33 242] TCPPacketReceiver::processPendingEvents(172.16.4.154:51712): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2017-11-06 17:10:33,244] - ==> [2017-11-06 17:10:33 243] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [2017-11-06 17:10:33,455] - TCPPacketReceiver::processPendingEvents(172.16.4.154:51713): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2017-11-06 17:10:33,455] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [2017-11-06 17:10:33,457] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2017-11-06 17:10:33,458] - onLoginAppShutDown()
  INFO root [2017-11-06 17:10:33,459] - Shutdowner::onShutdown: shutting down
  INFO root [2017-11-06 17:10:33,459] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2017-11-06 17:10:33,460] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [2017-11-06 17:10:33,460] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [2017-11-06 17:10:33,460] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [2017-11-06 17:10:33,461] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [2017-11-06 17:10:33,461] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [2017-11-06 17:10:33,461] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [2017-11-06 17:10:33,461] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [2017-11-06 17:10:33,476] - Script::uninstall(): is successfully!
 DEBUG root [2017-11-06 17:10:33,477] - ThreadPool::destroy(): starting size 2.
  WARN root [2017-11-06 17:10:33,778] - ThreadPool::destroy(): waiting for thread(2)[0x2185368,0x2185508,], try=1
 DEBUG root [2017-11-06 17:10:34,179] - TPThread::~TPThread(): 0x2185368
 DEBUG root [2017-11-06 17:10:34,179] - TPThread::~TPThread(): 0x2185508
 DEBUG root [2017-11-06 17:10:34,179] - ThreadPool::destroy(): successfully!
 DEBUG root [2017-11-06 17:10:34,187] - Bundle::destroyObjPool(): size 13.
 DEBUG root [2017-11-06 17:10:34,188] - Channel::destroyObjPool(): size 15.
 DEBUG root [2017-11-06 17:10:34,188] - TCPPacket::destroyObjPool(): size 12.
 DEBUG root [2017-11-06 17:10:34,188] - UDPPacket::destroyObjPool(): size 2.
 DEBUG root [2017-11-06 17:10:34,188] - EndPoint::destroyObjPool(): size 14.
 DEBUG root [2017-11-06 17:10:34,188] - Address::destroyObjPool(): size 0.
 DEBUG root [2017-11-06 17:10:34,188] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [2017-11-06 17:10:34,188] - UDPPacketReceiver::destroyObjPool(): size 0.
  INFO root [2017-11-06 17:10:34,188] - loginapp(9000) has shut down.
  INFO root [2017-11-06 17:10:39,662] - 
  INFO root [2017-11-06 17:10:39,662] - -----------------------------------------------------------------------------------------


  INFO root [2017-11-06 17:10:39,663] - Resmgr::initialize: KBE_ROOT=C:/Users/hyc/Desktop/kbengine/
  INFO root [2017-11-06 17:10:39,663] - Resmgr::initialize: KBE_RES_PATH=C:/Users/hyc/Desktop/kbengine/kbe/res/;C:/Users/hyc/Desktop/kbengine/llgame/;C:/Users/hyc/Desktop/kbengine/llgame/scripts/;C:/Users/hyc/Desktop/kbengine/llgame/res/
  INFO root [2017-11-06 17:10:39,663] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/hyc/Desktop/kbengine/kbe/bin/server/
  INFO root [2017-11-06 17:10:39,665] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 172.16.4.154 ( 172.16.4.154:20013 )
  INFO root [2017-11-06 17:10:39,666] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 172.16.4.154:20013, SOMAXCONN=128.
  INFO root [2017-11-06 17:10:39,666] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 172.16.4.154 ( 172.16.4.154:51774 )
  INFO root [2017-11-06 17:10:39,667] - NetworkInterface::recreateListeningSocket(INTERNAL): address 172.16.4.154:51774, SOMAXCONN=128.
  INFO root [2017-11-06 17:10:39,668] - server-configs:
  INFO root [2017-11-06 17:10:39,668] - 	internalAddr : 172.16.4.154:51774
  INFO root [2017-11-06 17:10:39,668] - 	externalAddr : 172.16.4.154:20013
  INFO root [2017-11-06 17:10:39,668] - 	componentID : 9000
  INFO root [2017-11-06 17:10:40,028] - Components::findLogger: found logger, addr:172.16.4.154:51768
  INFO root [2017-11-06 17:10:40,028] - Components::addComponent: reset orderLog, uid=1001!
  INFO root [2017-11-06 17:10:40,028] - Components::addComponent[logger], uid=1001, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2017-11-06 17:10:40,033] - ---- loginapp Version: 0.9.12. ScriptVersion: 0.1.0. Protocol: 75C8DA776708D1E89693088467E5A4FB. Config: Debug X86. Built: 16:13:51 Jul  7 2017. AppID: 9000. UID: 1001. PID: 93140 ----
  INFO root [2017-11-06 17:10:40,033] - Server : Intel(R) Core(TM) i5-4590 CPU @ 3.30GHz with 4194303kB RAM
  INFO root [2017-11-06 17:10:40,033] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2017-11-06 17:10:40,034] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2017-11-06 17:10:40,161] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2017-11-06 17:10:40,161] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2017-11-06 17:10:40,162] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2017-11-06 17:10:40,163] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2017-11-06 17:10:40,164] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2017-11-06 17:10:40,164] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2017-11-06 17:10:40,196] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2017-11-06 17:10:40,227] - Script::install(): is successfully, Python=(3.4.2 (default, Jul  7 2017, 16:13:50) [MSC v.1800 32 bit (Intel)])!
  INFO root [2017-11-06 17:10:40,227] - Script::install(): KBExtra is successfully!
S_INFO root [2017-11-06 17:10:40,229] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2017-11-06 17:10:40,230] - TelnetServer server is running on port 31000
  INFO root [2017-11-06 17:10:40,446] - ---- loginapp is running ----
 DEBUG root [2017-11-06 17:10:40,446] - Components::process(): Request for the process of identity...
  WARN root [2017-11-06 17:10:46,599] - Forwarding logs to logger[172.16.4.154:51768/0/0/0]...
 ERROR root [2017-11-06 17:22:56,745] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2017-11-06 17:22:56,746] - The following logs sent to logger failed:
  WARN root [2017-11-06 17:22:56,746] - ==> [2017-11-06 17:22:56 745] TCPPacketReceiver::processPendingEvents(172.16.4.154:51768): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2017-11-06 17:22:56,746] - ==> [2017-11-06 17:22:56 745] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [2017-11-06 17:22:56,987] - TCPPacketReceiver::processPendingEvents(172.16.4.154:51769): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2017-11-06 17:22:56,987] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [2017-11-06 17:22:56,988] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2017-11-06 17:22:56,988] - onLoginAppShutDown()
  INFO root [2017-11-06 17:22:56,988] - Shutdowner::onShutdown: shutting down
  INFO root [2017-11-06 17:22:56,988] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2017-11-06 17:22:56,989] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [2017-11-06 17:22:56,989] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [2017-11-06 17:22:56,989] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [2017-11-06 17:22:56,989] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [2017-11-06 17:22:56,989] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [2017-11-06 17:22:56,989] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [2017-11-06 17:22:56,989] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [2017-11-06 17:22:56,999] - Script::uninstall(): is successfully!
 DEBUG root [2017-11-06 17:22:57,000] - ThreadPool::destroy(): starting size 2.
  WARN root [2017-11-06 17:22:57,300] - ThreadPool::destroy(): waiting for thread(2)[0x1fd4438,0x1fd4c58,], try=1
 DEBUG root [2017-11-06 17:22:57,701] - TPThread::~TPThread(): 0x1fd4438
 DEBUG root [2017-11-06 17:22:57,701] - TPThread::~TPThread(): 0x1fd4c58
 DEBUG root [2017-11-06 17:22:57,701] - ThreadPool::destroy(): successfully!
 DEBUG root [2017-11-06 17:22:57,709] - Bundle::destroyObjPool(): size 13.
 DEBUG root [2017-11-06 17:22:57,709] - Channel::destroyObjPool(): size 15.
 DEBUG root [2017-11-06 17:22:57,710] - TCPPacket::destroyObjPool(): size 1.
 DEBUG root [2017-11-06 17:22:57,710] - UDPPacket::destroyObjPool(): size 2.
 DEBUG root [2017-11-06 17:22:57,710] - EndPoint::destroyObjPool(): size 14.
 DEBUG root [2017-11-06 17:22:57,710] - Address::destroyObjPool(): size 0.
 DEBUG root [2017-11-06 17:22:57,710] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [2017-11-06 17:22:57,710] - UDPPacketReceiver::destroyObjPool(): size 0.
  INFO root [2017-11-06 17:22:57,710] - loginapp(9000) has shut down.
  INFO root [2017-11-06 17:23:02,255] - 
  INFO root [2017-11-06 17:23:02,255] - -----------------------------------------------------------------------------------------


  INFO root [2017-11-06 17:23:02,256] - Resmgr::initialize: KBE_ROOT=C:/Users/hyc/Desktop/kbengine/
  INFO root [2017-11-06 17:23:02,256] - Resmgr::initialize: KBE_RES_PATH=C:/Users/hyc/Desktop/kbengine/kbe/res/;C:/Users/hyc/Desktop/kbengine/llgame/;C:/Users/hyc/Desktop/kbengine/llgame/scripts/;C:/Users/hyc/Desktop/kbengine/llgame/res/
  INFO root [2017-11-06 17:23:02,256] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/hyc/Desktop/kbengine/kbe/bin/server/
  INFO root [2017-11-06 17:23:02,257] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 172.16.4.154 ( 172.16.4.154:20013 )
  INFO root [2017-11-06 17:23:02,258] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 172.16.4.154:20013, SOMAXCONN=128.
  INFO root [2017-11-06 17:23:02,259] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 172.16.4.154 ( 172.16.4.154:51854 )
  INFO root [2017-11-06 17:23:02,260] - NetworkInterface::recreateListeningSocket(INTERNAL): address 172.16.4.154:51854, SOMAXCONN=128.
  INFO root [2017-11-06 17:23:02,260] - server-configs:
  INFO root [2017-11-06 17:23:02,260] - 	internalAddr : 172.16.4.154:51854
  INFO root [2017-11-06 17:23:02,260] - 	externalAddr : 172.16.4.154:20013
  INFO root [2017-11-06 17:23:02,260] - 	componentID : 9000
  INFO root [2017-11-06 17:23:02,595] - Components::findLogger: found logger, addr:172.16.4.154:51848
  INFO root [2017-11-06 17:23:02,595] - Components::addComponent: reset orderLog, uid=1001!
  INFO root [2017-11-06 17:23:02,595] - Components::addComponent[logger], uid=1001, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2017-11-06 17:23:02,599] - ---- loginapp Version: 0.9.12. ScriptVersion: 0.1.0. Protocol: 75C8DA776708D1E89693088467E5A4FB. Config: Debug X86. Built: 16:13:51 Jul  7 2017. AppID: 9000. UID: 1001. PID: 88704 ----
  INFO root [2017-11-06 17:23:02,600] - Server : Intel(R) Core(TM) i5-4590 CPU @ 3.30GHz with 4194303kB RAM
  INFO root [2017-11-06 17:23:02,600] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2017-11-06 17:23:02,600] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2017-11-06 17:23:02,727] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2017-11-06 17:23:02,727] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2017-11-06 17:23:02,728] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2017-11-06 17:23:02,729] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2017-11-06 17:23:02,729] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2017-11-06 17:23:02,730] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2017-11-06 17:23:02,761] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2017-11-06 17:23:02,791] - Script::install(): is successfully, Python=(3.4.2 (default, Jul  7 2017, 16:13:50) [MSC v.1800 32 bit (Intel)])!
  INFO root [2017-11-06 17:23:02,791] - Script::install(): KBExtra is successfully!
S_INFO root [2017-11-06 17:23:02,793] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2017-11-06 17:23:02,794] - TelnetServer server is running on port 31000
  INFO root [2017-11-06 17:23:03,008] - ---- loginapp is running ----
 DEBUG root [2017-11-06 17:23:03,009] - Components::process(): Request for the process of identity...
  WARN root [2017-11-06 17:23:09,162] - Forwarding logs to logger[172.16.4.154:51848/0/0/0]...
 ERROR root [2017-11-06 17:59:57,480] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2017-11-06 17:59:57,494] - The following logs sent to logger failed:
  WARN root [2017-11-06 17:59:57,494] - ==> [2017-11-06 17:59:57 478] TCPPacketReceiver::processPendingEvents(172.16.4.154:51848): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2017-11-06 17:59:57,495] - ==> [2017-11-06 17:59:57 480] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [2017-11-06 17:59:57,696] - TCPPacketReceiver::processPendingEvents(172.16.4.154:51850): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2017-11-06 17:59:57,697] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [2017-11-06 17:59:57,706] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2017-11-06 17:59:57,707] - onLoginAppShutDown()
  INFO root [2017-11-06 17:59:57,707] - Shutdowner::onShutdown: shutting down
  INFO root [2017-11-06 17:59:57,707] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2017-11-06 17:59:57,707] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [2017-11-06 17:59:57,707] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [2017-11-06 17:59:57,707] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [2017-11-06 17:59:57,707] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [2017-11-06 17:59:57,707] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [2017-11-06 17:59:57,707] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [2017-11-06 17:59:57,707] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [2017-11-06 17:59:57,716] - Script::uninstall(): is successfully!
 DEBUG root [2017-11-06 17:59:57,716] - ThreadPool::destroy(): starting size 2.
  WARN root [2017-11-06 17:59:58,018] - ThreadPool::destroy(): waiting for thread(2)[0x1da5360,0x1da5228,], try=1
 DEBUG root [2017-11-06 17:59:58,419] - TPThread::~TPThread(): 0x1da5360
 DEBUG root [2017-11-06 17:59:58,419] - TPThread::~TPThread(): 0x1da5228
 DEBUG root [2017-11-06 17:59:58,419] - ThreadPool::destroy(): successfully!
 DEBUG root [2017-11-06 17:59:58,427] - Bundle::destroyObjPool(): size 13.
 DEBUG root [2017-11-06 17:59:58,427] - Channel::destroyObjPool(): size 15.
 DEBUG root [2017-11-06 17:59:58,427] - TCPPacket::destroyObjPool(): size 1.
 DEBUG root [2017-11-06 17:59:58,427] - UDPPacket::destroyObjPool(): size 2.
 DEBUG root [2017-11-06 17:59:58,427] - EndPoint::destroyObjPool(): size 14.
 DEBUG root [2017-11-06 17:59:58,427] - Address::destroyObjPool(): size 0.
 DEBUG root [2017-11-06 17:59:58,428] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [2017-11-06 17:59:58,428] - UDPPacketReceiver::destroyObjPool(): size 0.
  INFO root [2017-11-06 17:59:58,428] - loginapp(9000) has shut down.
  INFO root [2017-11-06 18:00:03,420] - 
  INFO root [2017-11-06 18:00:03,420] - -----------------------------------------------------------------------------------------


  INFO root [2017-11-06 18:00:03,421] - Resmgr::initialize: KBE_ROOT=C:/Users/hyc/Desktop/kbengine/
  INFO root [2017-11-06 18:00:03,421] - Resmgr::initialize: KBE_RES_PATH=C:/Users/hyc/Desktop/kbengine/kbe/res/;C:/Users/hyc/Desktop/kbengine/llgame/;C:/Users/hyc/Desktop/kbengine/llgame/scripts/;C:/Users/hyc/Desktop/kbengine/llgame/res/
  INFO root [2017-11-06 18:00:03,421] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/hyc/Desktop/kbengine/kbe/bin/server/
  INFO root [2017-11-06 18:00:03,423] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 172.16.4.154 ( 172.16.4.154:20013 )
  INFO root [2017-11-06 18:00:03,424] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 172.16.4.154:20013, SOMAXCONN=128.
  INFO root [2017-11-06 18:00:03,425] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 172.16.4.154 ( 172.16.4.154:52582 )
  INFO root [2017-11-06 18:00:03,426] - NetworkInterface::recreateListeningSocket(INTERNAL): address 172.16.4.154:52582, SOMAXCONN=128.
  INFO root [2017-11-06 18:00:03,426] - server-configs:
  INFO root [2017-11-06 18:00:03,426] - 	internalAddr : 172.16.4.154:52582
  INFO root [2017-11-06 18:00:03,426] - 	externalAddr : 172.16.4.154:20013
  INFO root [2017-11-06 18:00:03,426] - 	componentID : 9000
  INFO root [2017-11-06 18:00:03,823] - Components::findLogger: found logger, addr:172.16.4.154:52577
  INFO root [2017-11-06 18:00:03,823] - Components::addComponent: reset orderLog, uid=1001!
  INFO root [2017-11-06 18:00:03,823] - Components::addComponent[logger], uid=1001, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2017-11-06 18:00:03,828] - ---- loginapp Version: 0.9.12. ScriptVersion: 0.1.0. Protocol: 75C8DA776708D1E89693088467E5A4FB. Config: Debug X86. Built: 16:13:51 Jul  7 2017. AppID: 9000. UID: 1001. PID: 3416 ----
  INFO root [2017-11-06 18:00:03,828] - Server : Intel(R) Core(TM) i5-4590 CPU @ 3.30GHz with 4194303kB RAM
  INFO root [2017-11-06 18:00:03,828] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2017-11-06 18:00:03,828] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2017-11-06 18:00:03,954] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2017-11-06 18:00:03,955] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2017-11-06 18:00:03,955] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2017-11-06 18:00:03,957] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2017-11-06 18:00:03,957] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2017-11-06 18:00:03,957] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2017-11-06 18:00:03,988] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2017-11-06 18:00:04,018] - Script::install(): is successfully, Python=(3.4.2 (default, Jul  7 2017, 16:13:50) [MSC v.1800 32 bit (Intel)])!
  INFO root [2017-11-06 18:00:04,019] - Script::install(): KBExtra is successfully!
S_INFO root [2017-11-06 18:00:04,021] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2017-11-06 18:00:04,021] - TelnetServer server is running on port 31000
  INFO root [2017-11-06 18:00:04,233] - ---- loginapp is running ----
 DEBUG root [2017-11-06 18:00:04,233] - Components::process(): Request for the process of identity...
  WARN root [2017-11-06 18:00:10,429] - Forwarding logs to logger[172.16.4.154:52577/0/0/0]...
 ERROR root [2017-11-06 18:27:04,890] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2017-11-06 18:27:04,891] - The following logs sent to logger failed:
  WARN root [2017-11-06 18:27:04,891] - ==> [2017-11-06 18:27:04 890] TCPPacketReceiver::processPendingEvents(172.16.4.154:52577): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2017-11-06 18:27:04,891] - ==> [2017-11-06 18:27:04 891] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [2017-11-06 18:27:05,103] - TCPPacketReceiver::processPendingEvents(172.16.4.154:52578): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2017-11-06 18:27:05,106] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [2017-11-06 18:27:05,110] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2017-11-06 18:27:05,110] - onLoginAppShutDown()
  INFO root [2017-11-06 18:27:05,110] - Shutdowner::onShutdown: shutting down
  INFO root [2017-11-06 18:27:05,110] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2017-11-06 18:27:05,110] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [2017-11-06 18:27:05,111] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [2017-11-06 18:27:05,111] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [2017-11-06 18:27:05,111] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [2017-11-06 18:27:05,111] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [2017-11-06 18:27:05,111] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [2017-11-06 18:27:05,111] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [2017-11-06 18:27:05,119] - Script::uninstall(): is successfully!
 DEBUG root [2017-11-06 18:27:05,119] - ThreadPool::destroy(): starting size 2.
  WARN root [2017-11-06 18:27:05,420] - ThreadPool::destroy(): waiting for thread(2)[0x2055cc0,0x2055028,], try=1
 DEBUG root [2017-11-06 18:27:05,822] - TPThread::~TPThread(): 0x2055cc0
 DEBUG root [2017-11-06 18:27:05,822] - TPThread::~TPThread(): 0x2055028
 DEBUG root [2017-11-06 18:27:05,822] - ThreadPool::destroy(): successfully!
 DEBUG root [2017-11-06 18:27:05,830] - Bundle::destroyObjPool(): size 13.
 DEBUG root [2017-11-06 18:27:05,830] - Channel::destroyObjPool(): size 15.
 DEBUG root [2017-11-06 18:27:05,831] - TCPPacket::destroyObjPool(): size 1.
 DEBUG root [2017-11-06 18:27:05,831] - UDPPacket::destroyObjPool(): size 2.
 DEBUG root [2017-11-06 18:27:05,831] - EndPoint::destroyObjPool(): size 14.
 DEBUG root [2017-11-06 18:27:05,831] - Address::destroyObjPool(): size 0.
 DEBUG root [2017-11-06 18:27:05,831] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [2017-11-06 18:27:05,831] - UDPPacketReceiver::destroyObjPool(): size 0.
  INFO root [2017-11-06 18:27:05,831] - loginapp(9000) has shut down.
  INFO root [2017-11-06 18:27:11,158] - 
  INFO root [2017-11-06 18:27:11,158] - -----------------------------------------------------------------------------------------


  INFO root [2017-11-06 18:27:11,159] - Resmgr::initialize: KBE_ROOT=C:/Users/hyc/Desktop/kbengine/
  INFO root [2017-11-06 18:27:11,159] - Resmgr::initialize: KBE_RES_PATH=C:/Users/hyc/Desktop/kbengine/kbe/res/;C:/Users/hyc/Desktop/kbengine/llgame/;C:/Users/hyc/Desktop/kbengine/llgame/scripts/;C:/Users/hyc/Desktop/kbengine/llgame/res/
  INFO root [2017-11-06 18:27:11,159] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/hyc/Desktop/kbengine/kbe/bin/server/
  INFO root [2017-11-06 18:27:11,161] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 172.16.4.154 ( 172.16.4.154:20013 )
  INFO root [2017-11-06 18:27:11,162] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 172.16.4.154:20013, SOMAXCONN=128.
  INFO root [2017-11-06 18:27:11,162] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 172.16.4.154 ( 172.16.4.154:52757 )
  INFO root [2017-11-06 18:27:11,163] - NetworkInterface::recreateListeningSocket(INTERNAL): address 172.16.4.154:52757, SOMAXCONN=128.
  INFO root [2017-11-06 18:27:11,163] - server-configs:
  INFO root [2017-11-06 18:27:11,164] - 	internalAddr : 172.16.4.154:52757
  INFO root [2017-11-06 18:27:11,164] - 	externalAddr : 172.16.4.154:20013
  INFO root [2017-11-06 18:27:11,164] - 	componentID : 9000
  INFO root [2017-11-06 18:27:11,617] - Components::findLogger: found logger, addr:172.16.4.154:52752
  INFO root [2017-11-06 18:27:11,618] - Components::addComponent: reset orderLog, uid=1001!
  INFO root [2017-11-06 18:27:11,618] - Components::addComponent[logger], uid=1001, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2017-11-06 18:27:11,622] - ---- loginapp Version: 0.9.12. ScriptVersion: 0.1.0. Protocol: 75C8DA776708D1E89693088467E5A4FB. Config: Debug X86. Built: 16:13:51 Jul  7 2017. AppID: 9000. UID: 1001. PID: 94792 ----
  INFO root [2017-11-06 18:27:11,623] - Server : Intel(R) Core(TM) i5-4590 CPU @ 3.30GHz with 4194303kB RAM
  INFO root [2017-11-06 18:27:11,623] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2017-11-06 18:27:11,623] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2017-11-06 18:27:11,802] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2017-11-06 18:27:11,802] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2017-11-06 18:27:11,802] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2017-11-06 18:27:11,805] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2017-11-06 18:27:11,805] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2017-11-06 18:27:11,805] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2017-11-06 18:27:11,869] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2017-11-06 18:27:11,914] - Script::install(): is successfully, Python=(3.4.2 (default, Jul  7 2017, 16:13:50) [MSC v.1800 32 bit (Intel)])!
  INFO root [2017-11-06 18:27:11,914] - Script::install(): KBExtra is successfully!
S_INFO root [2017-11-06 18:27:11,916] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2017-11-06 18:27:11,916] - TelnetServer server is running on port 31000
  INFO root [2017-11-06 18:27:12,204] - ---- loginapp is running ----
 DEBUG root [2017-11-06 18:27:12,214] - Components::process(): Request for the process of identity...
  WARN root [2017-11-06 18:27:12,555] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20305, errorno=10048
  WARN root [2017-11-06 18:27:18,727] - Forwarding logs to logger[172.16.4.154:52752/0/0/0]...
 ERROR root [2017-11-06 19:49:06,464] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2017-11-06 19:49:06,484] - The following logs sent to logger failed:
  WARN root [2017-11-06 19:49:06,485] - ==> [2017-11-06 19:49:06 461] TCPPacketReceiver::processPendingEvents(172.16.4.154:52752): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2017-11-06 19:49:06,485] - ==> [2017-11-06 19:49:06 464] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [2017-11-06 19:49:06,675] - TCPPacketReceiver::processPendingEvents(172.16.4.154:52754): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2017-11-06 19:49:06,679] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [2017-11-06 19:49:06,690] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2017-11-06 19:49:06,690] - onLoginAppShutDown()
  INFO root [2017-11-06 19:49:06,690] - Shutdowner::onShutdown: shutting down
  INFO root [2017-11-06 19:49:06,690] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2017-11-06 19:49:06,691] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [2017-11-06 19:49:06,691] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [2017-11-06 19:49:06,691] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [2017-11-06 19:49:06,691] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [2017-11-06 19:49:06,691] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [2017-11-06 19:49:06,691] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [2017-11-06 19:49:06,691] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [2017-11-06 19:49:06,701] - Script::uninstall(): is successfully!
 DEBUG root [2017-11-06 19:49:06,702] - ThreadPool::destroy(): starting size 2.
  WARN root [2017-11-06 19:49:07,002] - ThreadPool::destroy(): waiting for thread(2)[0x3f5bf0,0x3f5c58,], try=1
 DEBUG root [2017-11-06 19:49:07,404] - TPThread::~TPThread(): 0x3f5bf0
 DEBUG root [2017-11-06 19:49:07,404] - TPThread::~TPThread(): 0x3f5c58
 DEBUG root [2017-11-06 19:49:07,404] - ThreadPool::destroy(): successfully!
 DEBUG root [2017-11-06 19:49:07,412] - Bundle::destroyObjPool(): size 13.
 DEBUG root [2017-11-06 19:49:07,412] - Channel::destroyObjPool(): size 15.
 DEBUG root [2017-11-06 19:49:07,412] - TCPPacket::destroyObjPool(): size 1.
 DEBUG root [2017-11-06 19:49:07,412] - UDPPacket::destroyObjPool(): size 15.
 DEBUG root [2017-11-06 19:49:07,412] - EndPoint::destroyObjPool(): size 14.
 DEBUG root [2017-11-06 19:49:07,413] - Address::destroyObjPool(): size 0.
 DEBUG root [2017-11-06 19:49:07,413] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [2017-11-06 19:49:07,413] - UDPPacketReceiver::destroyObjPool(): size 0.
  INFO root [2017-11-06 19:49:07,413] - loginapp(9000) has shut down.
  INFO root [2017-11-06 19:53:13,396] - 
  INFO root [2017-11-06 19:53:13,397] - -----------------------------------------------------------------------------------------


  INFO root [2017-11-06 19:53:13,397] - Resmgr::initialize: KBE_ROOT=C:/Users/hyc/Desktop/kbengine/
  INFO root [2017-11-06 19:53:13,397] - Resmgr::initialize: KBE_RES_PATH=C:/Users/hyc/Desktop/kbengine/kbe/res/;C:/Users/hyc/Desktop/kbengine/llgame/;C:/Users/hyc/Desktop/kbengine/llgame/scripts/;C:/Users/hyc/Desktop/kbengine/llgame/res/
  INFO root [2017-11-06 19:53:13,398] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/hyc/Desktop/kbengine/kbe/bin/server/
  INFO root [2017-11-06 19:53:13,399] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 172.16.4.154 ( 172.16.4.154:20013 )
  INFO root [2017-11-06 19:53:13,400] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 172.16.4.154:20013, SOMAXCONN=128.
  INFO root [2017-11-06 19:53:13,401] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 172.16.4.154 ( 172.16.4.154:53050 )
  INFO root [2017-11-06 19:53:13,402] - NetworkInterface::recreateListeningSocket(INTERNAL): address 172.16.4.154:53050, SOMAXCONN=128.
  INFO root [2017-11-06 19:53:13,402] - server-configs:
  INFO root [2017-11-06 19:53:13,402] - 	internalAddr : 172.16.4.154:53050
  INFO root [2017-11-06 19:53:13,402] - 	externalAddr : 172.16.4.154:20013
  INFO root [2017-11-06 19:53:13,402] - 	componentID : 9000
  INFO root [2017-11-06 19:53:13,787] - Components::findLogger: found logger, addr:172.16.4.154:53043
  INFO root [2017-11-06 19:53:13,787] - Components::addComponent: reset orderLog, uid=1001!
  INFO root [2017-11-06 19:53:13,787] - Components::addComponent[logger], uid=1001, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2017-11-06 19:53:13,792] - ---- loginapp Version: 0.9.12. ScriptVersion: 0.1.0. Protocol: 75C8DA776708D1E89693088467E5A4FB. Config: Debug X86. Built: 16:13:51 Jul  7 2017. AppID: 9000. UID: 1001. PID: 93304 ----
  INFO root [2017-11-06 19:53:13,792] - Server : Intel(R) Core(TM) i5-4590 CPU @ 3.30GHz with 4194303kB RAM
  INFO root [2017-11-06 19:53:13,793] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2017-11-06 19:53:13,793] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2017-11-06 19:53:13,920] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2017-11-06 19:53:13,920] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2017-11-06 19:53:13,920] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2017-11-06 19:53:13,922] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2017-11-06 19:53:13,922] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2017-11-06 19:53:13,923] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2017-11-06 19:53:13,955] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2017-11-06 19:53:13,985] - Script::install(): is successfully, Python=(3.4.2 (default, Jul  7 2017, 16:13:50) [MSC v.1800 32 bit (Intel)])!
  INFO root [2017-11-06 19:53:13,985] - Script::install(): KBExtra is successfully!
S_INFO root [2017-11-06 19:53:13,988] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2017-11-06 19:53:13,988] - TelnetServer server is running on port 31000
  INFO root [2017-11-06 19:53:14,201] - ---- loginapp is running ----
 DEBUG root [2017-11-06 19:53:14,201] - Components::process(): Request for the process of identity...
  WARN root [2017-11-06 19:53:20,375] - Forwarding logs to logger[172.16.4.154:53043/0/0/0]...
 ERROR root [2017-11-06 19:53:38,444] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2017-11-06 19:53:38,449] - The following logs sent to logger failed:
  WARN root [2017-11-06 19:53:38,449] - ==> [2017-11-06 19:53:38 443] TCPPacketReceiver::processPendingEvents(172.16.4.154:53043): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2017-11-06 19:53:38,449] - ==> [2017-11-06 19:53:38 444] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [2017-11-06 19:53:38,635] - TCPPacketReceiver::processPendingEvents(172.16.4.154:53047): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2017-11-06 19:53:38,638] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [2017-11-06 19:53:38,645] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2017-11-06 19:53:38,645] - onLoginAppShutDown()
  INFO root [2017-11-06 19:53:38,645] - Shutdowner::onShutdown: shutting down
  INFO root [2017-11-06 19:53:38,645] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2017-11-06 19:53:38,646] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [2017-11-06 19:53:38,646] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [2017-11-06 19:53:38,646] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [2017-11-06 19:53:38,646] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [2017-11-06 19:53:38,646] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [2017-11-06 19:53:38,646] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [2017-11-06 19:53:38,646] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [2017-11-06 19:53:38,655] - Script::uninstall(): is successfully!
 DEBUG root [2017-11-06 19:53:38,655] - ThreadPool::destroy(): starting size 2.
  WARN root [2017-11-06 19:53:38,956] - ThreadPool::destroy(): waiting for thread(2)[0xa750f8,0xa75230,], try=1
 DEBUG root [2017-11-06 19:53:39,357] - TPThread::~TPThread(): 0xa750f8
 DEBUG root [2017-11-06 19:53:39,357] - TPThread::~TPThread(): 0xa75230
 DEBUG root [2017-11-06 19:53:39,357] - ThreadPool::destroy(): successfully!
 DEBUG root [2017-11-06 19:53:39,365] - Bundle::destroyObjPool(): size 13.
 DEBUG root [2017-11-06 19:53:39,365] - Channel::destroyObjPool(): size 15.
 DEBUG root [2017-11-06 19:53:39,365] - TCPPacket::destroyObjPool(): size 12.
 DEBUG root [2017-11-06 19:53:39,365] - UDPPacket::destroyObjPool(): size 1.
 DEBUG root [2017-11-06 19:53:39,365] - EndPoint::destroyObjPool(): size 14.
 DEBUG root [2017-11-06 19:53:39,366] - Address::destroyObjPool(): size 0.
 DEBUG root [2017-11-06 19:53:39,366] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [2017-11-06 19:53:39,366] - UDPPacketReceiver::destroyObjPool(): size 0.
  INFO root [2017-11-06 19:53:39,366] - loginapp(9000) has shut down.
  INFO root [2017-11-06 19:53:41,573] - 
  INFO root [2017-11-06 19:53:41,573] - -----------------------------------------------------------------------------------------


  INFO root [2017-11-06 19:53:41,574] - Resmgr::initialize: KBE_ROOT=C:/Users/hyc/Desktop/kbengine/
  INFO root [2017-11-06 19:53:41,574] - Resmgr::initialize: KBE_RES_PATH=C:/Users/hyc/Desktop/kbengine/kbe/res/;C:/Users/hyc/Desktop/kbengine/llgame/;C:/Users/hyc/Desktop/kbengine/llgame/scripts/;C:/Users/hyc/Desktop/kbengine/llgame/res/
  INFO root [2017-11-06 19:53:41,574] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/hyc/Desktop/kbengine/kbe/bin/server/
  INFO root [2017-11-06 19:53:41,576] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 172.16.4.154 ( 172.16.4.154:20013 )
  INFO root [2017-11-06 19:53:41,577] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 172.16.4.154:20013, SOMAXCONN=128.
  INFO root [2017-11-06 19:53:41,578] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 172.16.4.154 ( 172.16.4.154:53092 )
  INFO root [2017-11-06 19:53:41,579] - NetworkInterface::recreateListeningSocket(INTERNAL): address 172.16.4.154:53092, SOMAXCONN=128.
  INFO root [2017-11-06 19:53:41,579] - server-configs:
  INFO root [2017-11-06 19:53:41,579] - 	internalAddr : 172.16.4.154:53092
  INFO root [2017-11-06 19:53:41,579] - 	externalAddr : 172.16.4.154:20013
  INFO root [2017-11-06 19:53:41,579] - 	componentID : 9000
  INFO root [2017-11-06 19:53:41,984] - Components::findLogger: found logger, addr:172.16.4.154:53085
  INFO root [2017-11-06 19:53:41,984] - Components::addComponent: reset orderLog, uid=1001!
  INFO root [2017-11-06 19:53:41,984] - Components::addComponent[logger], uid=1001, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2017-11-06 19:53:41,988] - ---- loginapp Version: 0.9.12. ScriptVersion: 0.1.0. Protocol: 75C8DA776708D1E89693088467E5A4FB. Config: Debug X86. Built: 16:13:51 Jul  7 2017. AppID: 9000. UID: 1001. PID: 93312 ----
  INFO root [2017-11-06 19:53:41,989] - Server : Intel(R) Core(TM) i5-4590 CPU @ 3.30GHz with 4194303kB RAM
  INFO root [2017-11-06 19:53:41,989] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2017-11-06 19:53:41,989] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2017-11-06 19:53:42,116] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2017-11-06 19:53:42,117] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2017-11-06 19:53:42,117] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2017-11-06 19:53:42,119] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2017-11-06 19:53:42,119] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2017-11-06 19:53:42,119] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2017-11-06 19:53:42,151] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2017-11-06 19:53:42,182] - Script::install(): is successfully, Python=(3.4.2 (default, Jul  7 2017, 16:13:50) [MSC v.1800 32 bit (Intel)])!
  INFO root [2017-11-06 19:53:42,182] - Script::install(): KBExtra is successfully!
S_INFO root [2017-11-06 19:53:42,185] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2017-11-06 19:53:42,185] - TelnetServer server is running on port 31000
  INFO root [2017-11-06 19:53:42,397] - ---- loginapp is running ----
 DEBUG root [2017-11-06 19:53:42,397] - Components::process(): Request for the process of identity...
  WARN root [2017-11-06 19:53:48,552] - Forwarding logs to logger[172.16.4.154:53085/0/0/0]...
 ERROR root [2017-11-06 19:55:12,137] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2017-11-06 19:55:12,148] - The following logs sent to logger failed:
  WARN root [2017-11-06 19:55:12,148] - ==> [2017-11-06 19:55:12 137] TCPPacketReceiver::processPendingEvents(172.16.4.154:53085): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2017-11-06 19:55:12,148] - ==> [2017-11-06 19:55:12 137] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [2017-11-06 19:55:12,340] - TCPPacketReceiver::processPendingEvents(172.16.4.154:53087): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2017-11-06 19:55:12,340] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [2017-11-06 19:55:12,345] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2017-11-06 19:55:12,347] - onLoginAppShutDown()
  INFO root [2017-11-06 19:55:12,348] - Shutdowner::onShutdown: shutting down
  INFO root [2017-11-06 19:55:12,348] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2017-11-06 19:55:12,348] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [2017-11-06 19:55:12,348] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [2017-11-06 19:55:12,348] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [2017-11-06 19:55:12,348] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [2017-11-06 19:55:12,348] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [2017-11-06 19:55:12,348] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [2017-11-06 19:55:12,348] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [2017-11-06 19:55:12,357] - Script::uninstall(): is successfully!
 DEBUG root [2017-11-06 19:55:12,357] - ThreadPool::destroy(): starting size 2.
  WARN root [2017-11-06 19:55:12,659] - ThreadPool::destroy(): waiting for thread(2)[0x5650f8,0x565368,], try=1
 DEBUG root [2017-11-06 19:55:13,060] - TPThread::~TPThread(): 0x5650f8
 DEBUG root [2017-11-06 19:55:13,060] - TPThread::~TPThread(): 0x565368
 DEBUG root [2017-11-06 19:55:13,060] - ThreadPool::destroy(): successfully!
 DEBUG root [2017-11-06 19:55:13,068] - Bundle::destroyObjPool(): size 13.
 DEBUG root [2017-11-06 19:55:13,068] - Channel::destroyObjPool(): size 13.
 DEBUG root [2017-11-06 19:55:13,068] - TCPPacket::destroyObjPool(): size 12.
 DEBUG root [2017-11-06 19:55:13,068] - UDPPacket::destroyObjPool(): size 1.
 DEBUG root [2017-11-06 19:55:13,068] - EndPoint::destroyObjPool(): size 12.
 DEBUG root [2017-11-06 19:55:13,068] - Address::destroyObjPool(): size 0.
 DEBUG root [2017-11-06 19:55:13,068] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [2017-11-06 19:55:13,068] - UDPPacketReceiver::destroyObjPool(): size 0.
  INFO root [2017-11-06 19:55:13,068] - loginapp(9000) has shut down.
  INFO root [2017-11-06 19:55:18,625] - 
  INFO root [2017-11-06 19:55:18,626] - -----------------------------------------------------------------------------------------


  INFO root [2017-11-06 19:55:18,626] - Resmgr::initialize: KBE_ROOT=C:/Users/hyc/Desktop/kbengine/
  INFO root [2017-11-06 19:55:18,627] - Resmgr::initialize: KBE_RES_PATH=C:/Users/hyc/Desktop/kbengine/kbe/res/;C:/Users/hyc/Desktop/kbengine/llgame/;C:/Users/hyc/Desktop/kbengine/llgame/scripts/;C:/Users/hyc/Desktop/kbengine/llgame/res/
  INFO root [2017-11-06 19:55:18,627] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/hyc/Desktop/kbengine/kbe/bin/server/
  INFO root [2017-11-06 19:55:18,630] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 172.16.4.154 ( 172.16.4.154:20013 )
  INFO root [2017-11-06 19:55:18,631] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 172.16.4.154:20013, SOMAXCONN=128.
  INFO root [2017-11-06 19:55:18,632] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 172.16.4.154 ( 172.16.4.154:53146 )
  INFO root [2017-11-06 19:55:18,633] - NetworkInterface::recreateListeningSocket(INTERNAL): address 172.16.4.154:53146, SOMAXCONN=128.
  INFO root [2017-11-06 19:55:18,633] - server-configs:
  INFO root [2017-11-06 19:55:18,633] - 	internalAddr : 172.16.4.154:53146
  INFO root [2017-11-06 19:55:18,633] - 	externalAddr : 172.16.4.154:20013
  INFO root [2017-11-06 19:55:18,633] - 	componentID : 9000
  INFO root [2017-11-06 19:55:19,126] - Components::findLogger: found logger, addr:172.16.4.154:53141
  INFO root [2017-11-06 19:55:19,126] - Components::addComponent: reset orderLog, uid=1001!
  INFO root [2017-11-06 19:55:19,127] - Components::addComponent[logger], uid=1001, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2017-11-06 19:55:19,131] - ---- loginapp Version: 0.9.12. ScriptVersion: 0.1.0. Protocol: 75C8DA776708D1E89693088467E5A4FB. Config: Debug X86. Built: 16:13:51 Jul  7 2017. AppID: 9000. UID: 1001. PID: 95256 ----
  INFO root [2017-11-06 19:55:19,131] - Server : Intel(R) Core(TM) i5-4590 CPU @ 3.30GHz with 4194303kB RAM
  INFO root [2017-11-06 19:55:19,131] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2017-11-06 19:55:19,132] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2017-11-06 19:55:19,269] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2017-11-06 19:55:19,269] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2017-11-06 19:55:19,270] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2017-11-06 19:55:19,271] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2017-11-06 19:55:19,271] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2017-11-06 19:55:19,272] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2017-11-06 19:55:19,304] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2017-11-06 19:55:19,336] - Script::install(): is successfully, Python=(3.4.2 (default, Jul  7 2017, 16:13:50) [MSC v.1800 32 bit (Intel)])!
  INFO root [2017-11-06 19:55:19,336] - Script::install(): KBExtra is successfully!
S_INFO root [2017-11-06 19:55:19,338] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2017-11-06 19:55:19,339] - TelnetServer server is running on port 31000
  INFO root [2017-11-06 19:55:19,564] - ---- loginapp is running ----
 DEBUG root [2017-11-06 19:55:19,564] - Components::process(): Request for the process of identity...
  WARN root [2017-11-06 19:55:19,906] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20862, errorno=10048
  WARN root [2017-11-06 19:55:26,054] - Forwarding logs to logger[172.16.4.154:53141/0/0/0]...
 ERROR root [2017-11-06 19:56:26,473] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2017-11-06 19:56:26,479] - The following logs sent to logger failed:
  WARN root [2017-11-06 19:56:26,479] - ==> [2017-11-06 19:56:26 473] TCPPacketReceiver::processPendingEvents(172.16.4.154:53141): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2017-11-06 19:56:26,479] - ==> [2017-11-06 19:56:26 474] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [2017-11-06 19:56:26,685] - TCPPacketReceiver::processPendingEvents(172.16.4.154:53142): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2017-11-06 19:56:26,687] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [2017-11-06 19:56:26,689] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2017-11-06 19:56:26,689] - onLoginAppShutDown()
  INFO root [2017-11-06 19:56:26,689] - Shutdowner::onShutdown: shutting down
  INFO root [2017-11-06 19:56:26,689] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2017-11-06 19:56:26,690] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [2017-11-06 19:56:26,690] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [2017-11-06 19:56:26,690] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [2017-11-06 19:56:26,690] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [2017-11-06 19:56:26,690] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [2017-11-06 19:56:26,690] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [2017-11-06 19:56:26,690] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [2017-11-06 19:56:26,702] - Script::uninstall(): is successfully!
 DEBUG root [2017-11-06 19:56:26,702] - ThreadPool::destroy(): starting size 2.
  WARN root [2017-11-06 19:56:27,004] - ThreadPool::destroy(): waiting for thread(2)[0x8c54a0,0x8c5778,], try=1
 DEBUG root [2017-11-06 19:56:27,405] - TPThread::~TPThread(): 0x8c54a0
 DEBUG root [2017-11-06 19:56:27,405] - TPThread::~TPThread(): 0x8c5778
 DEBUG root [2017-11-06 19:56:27,405] - ThreadPool::destroy(): successfully!
 DEBUG root [2017-11-06 19:56:27,413] - Bundle::destroyObjPool(): size 29.
 DEBUG root [2017-11-06 19:56:27,413] - Channel::destroyObjPool(): size 13.
 DEBUG root [2017-11-06 19:56:27,413] - TCPPacket::destroyObjPool(): size 12.
 DEBUG root [2017-11-06 19:56:27,413] - UDPPacket::destroyObjPool(): size 1.
 DEBUG root [2017-11-06 19:56:27,413] - EndPoint::destroyObjPool(): size 12.
 DEBUG root [2017-11-06 19:56:27,413] - Address::destroyObjPool(): size 0.
 DEBUG root [2017-11-06 19:56:27,413] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [2017-11-06 19:56:27,413] - UDPPacketReceiver::destroyObjPool(): size 0.
  INFO root [2017-11-06 19:56:27,414] - loginapp(9000) has shut down.
  INFO root [2017-11-06 19:57:26,439] - 
  INFO root [2017-11-06 19:57:26,439] - -----------------------------------------------------------------------------------------


  INFO root [2017-11-06 19:57:26,440] - Resmgr::initialize: KBE_ROOT=C:/Users/hyc/Desktop/kbengine/
  INFO root [2017-11-06 19:57:26,440] - Resmgr::initialize: KBE_RES_PATH=C:/Users/hyc/Desktop/kbengine/kbe/res/;C:/Users/hyc/Desktop/kbengine/llgame/;C:/Users/hyc/Desktop/kbengine/llgame/scripts/;C:/Users/hyc/Desktop/kbengine/llgame/res/
  INFO root [2017-11-06 19:57:26,440] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/hyc/Desktop/kbengine/kbe/bin/server/
  INFO root [2017-11-06 19:57:26,442] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 172.16.4.154 ( 172.16.4.154:20013 )
  INFO root [2017-11-06 19:57:26,443] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 172.16.4.154:20013, SOMAXCONN=128.
  INFO root [2017-11-06 19:57:26,444] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 172.16.4.154 ( 172.16.4.154:53202 )
  INFO root [2017-11-06 19:57:26,445] - NetworkInterface::recreateListeningSocket(INTERNAL): address 172.16.4.154:53202, SOMAXCONN=128.
  INFO root [2017-11-06 19:57:26,445] - server-configs:
  INFO root [2017-11-06 19:57:26,445] - 	internalAddr : 172.16.4.154:53202
  INFO root [2017-11-06 19:57:26,445] - 	externalAddr : 172.16.4.154:20013
  INFO root [2017-11-06 19:57:26,445] - 	componentID : 9000
  INFO root [2017-11-06 19:57:26,851] - Components::findLogger: found logger, addr:172.16.4.154:53196
  INFO root [2017-11-06 19:57:26,851] - Components::addComponent: reset orderLog, uid=1001!
  INFO root [2017-11-06 19:57:26,851] - Components::addComponent[logger], uid=1001, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2017-11-06 19:57:26,855] - ---- loginapp Version: 0.9.12. ScriptVersion: 0.1.0. Protocol: 75C8DA776708D1E89693088467E5A4FB. Config: Debug X86. Built: 16:13:51 Jul  7 2017. AppID: 9000. UID: 1001. PID: 96024 ----
  INFO root [2017-11-06 19:57:26,856] - Server : Intel(R) Core(TM) i5-4590 CPU @ 3.30GHz with 4194303kB RAM
  INFO root [2017-11-06 19:57:26,856] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2017-11-06 19:57:26,856] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2017-11-06 19:57:26,983] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2017-11-06 19:57:26,984] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2017-11-06 19:57:26,984] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2017-11-06 19:57:26,986] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2017-11-06 19:57:26,986] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2017-11-06 19:57:26,986] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2017-11-06 19:57:27,019] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2017-11-06 19:57:27,050] - Script::install(): is successfully, Python=(3.4.2 (default, Jul  7 2017, 16:13:50) [MSC v.1800 32 bit (Intel)])!
  INFO root [2017-11-06 19:57:27,050] - Script::install(): KBExtra is successfully!
S_INFO root [2017-11-06 19:57:27,052] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2017-11-06 19:57:27,052] - TelnetServer server is running on port 31000
  INFO root [2017-11-06 19:57:27,270] - ---- loginapp is running ----
 DEBUG root [2017-11-06 19:57:27,270] - Components::process(): Request for the process of identity...
  WARN root [2017-11-06 19:57:33,441] - Forwarding logs to logger[172.16.4.154:53196/0/0/0]...
 ERROR root [2017-11-06 20:06:37,529] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2017-11-06 20:06:37,540] - The following logs sent to logger failed:
  WARN root [2017-11-06 20:06:37,541] - ==> [2017-11-06 20:06:37 529] TCPPacketReceiver::processPendingEvents(172.16.4.154:53196): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2017-11-06 20:06:37,541] - ==> [2017-11-06 20:06:37 533] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [2017-11-06 20:06:37,742] - TCPPacketReceiver::processPendingEvents(172.16.4.154:53197): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2017-11-06 20:06:37,749] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [2017-11-06 20:06:37,750] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2017-11-06 20:06:37,751] - onLoginAppShutDown()
  INFO root [2017-11-06 20:06:37,751] - Shutdowner::onShutdown: shutting down
  INFO root [2017-11-06 20:06:37,751] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2017-11-06 20:06:37,751] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [2017-11-06 20:06:37,751] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [2017-11-06 20:06:37,751] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [2017-11-06 20:06:37,751] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [2017-11-06 20:06:37,751] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [2017-11-06 20:06:37,751] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [2017-11-06 20:06:37,751] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [2017-11-06 20:06:37,763] - Script::uninstall(): is successfully!
 DEBUG root [2017-11-06 20:06:37,764] - ThreadPool::destroy(): starting size 2.
  WARN root [2017-11-06 20:06:38,064] - ThreadPool::destroy(): waiting for thread(2)[0x2025c58,0x2025cc0,], try=1
 DEBUG root [2017-11-06 20:06:38,467] - TPThread::~TPThread(): 0x2025c58
 DEBUG root [2017-11-06 20:06:38,467] - TPThread::~TPThread(): 0x2025cc0
 DEBUG root [2017-11-06 20:06:38,467] - ThreadPool::destroy(): successfully!
 DEBUG root [2017-11-06 20:06:38,475] - Bundle::destroyObjPool(): size 15.
 DEBUG root [2017-11-06 20:06:38,475] - Channel::destroyObjPool(): size 15.
 DEBUG root [2017-11-06 20:06:38,475] - TCPPacket::destroyObjPool(): size 15.
 DEBUG root [2017-11-06 20:06:38,475] - UDPPacket::destroyObjPool(): size 2.
 DEBUG root [2017-11-06 20:06:38,475] - EndPoint::destroyObjPool(): size 14.
 DEBUG root [2017-11-06 20:06:38,476] - Address::destroyObjPool(): size 0.
 DEBUG root [2017-11-06 20:06:38,476] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [2017-11-06 20:06:38,476] - UDPPacketReceiver::destroyObjPool(): size 0.
  INFO root [2017-11-06 20:06:38,476] - loginapp(9000) has shut down.
  INFO root [2017-11-06 20:06:43,854] - 
  INFO root [2017-11-06 20:06:43,855] - -----------------------------------------------------------------------------------------


  INFO root [2017-11-06 20:06:43,855] - Resmgr::initialize: KBE_ROOT=C:/Users/hyc/Desktop/kbengine/
  INFO root [2017-11-06 20:06:43,855] - Resmgr::initialize: KBE_RES_PATH=C:/Users/hyc/Desktop/kbengine/kbe/res/;C:/Users/hyc/Desktop/kbengine/llgame/;C:/Users/hyc/Desktop/kbengine/llgame/scripts/;C:/Users/hyc/Desktop/kbengine/llgame/res/
  INFO root [2017-11-06 20:06:43,856] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/hyc/Desktop/kbengine/kbe/bin/server/
  INFO root [2017-11-06 20:06:43,857] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 172.16.4.154 ( 172.16.4.154:20013 )
  INFO root [2017-11-06 20:06:43,858] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 172.16.4.154:20013, SOMAXCONN=128.
  INFO root [2017-11-06 20:06:43,859] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 172.16.4.154 ( 172.16.4.154:53271 )
  INFO root [2017-11-06 20:06:43,860] - NetworkInterface::recreateListeningSocket(INTERNAL): address 172.16.4.154:53271, SOMAXCONN=128.
  INFO root [2017-11-06 20:06:43,860] - server-configs:
  INFO root [2017-11-06 20:06:43,860] - 	internalAddr : 172.16.4.154:53271
  INFO root [2017-11-06 20:06:43,860] - 	externalAddr : 172.16.4.154:20013
  INFO root [2017-11-06 20:06:43,860] - 	componentID : 9000
  INFO root [2017-11-06 20:06:44,290] - Components::findLogger: found logger, addr:172.16.4.154:53266
  INFO root [2017-11-06 20:06:44,290] - Components::addComponent: reset orderLog, uid=1001!
  INFO root [2017-11-06 20:06:44,290] - Components::addComponent[logger], uid=1001, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2017-11-06 20:06:44,294] - ---- loginapp Version: 0.9.12. ScriptVersion: 0.1.0. Protocol: 75C8DA776708D1E89693088467E5A4FB. Config: Debug X86. Built: 16:13:51 Jul  7 2017. AppID: 9000. UID: 1001. PID: 96544 ----
  INFO root [2017-11-06 20:06:44,294] - Server : Intel(R) Core(TM) i5-4590 CPU @ 3.30GHz with 4194303kB RAM
  INFO root [2017-11-06 20:06:44,295] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2017-11-06 20:06:44,295] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2017-11-06 20:06:44,424] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2017-11-06 20:06:44,424] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2017-11-06 20:06:44,424] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2017-11-06 20:06:44,428] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2017-11-06 20:06:44,428] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2017-11-06 20:06:44,428] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2017-11-06 20:06:44,464] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2017-11-06 20:06:44,498] - Script::install(): is successfully, Python=(3.4.2 (default, Jul  7 2017, 16:13:50) [MSC v.1800 32 bit (Intel)])!
  INFO root [2017-11-06 20:06:44,498] - Script::install(): KBExtra is successfully!
S_INFO root [2017-11-06 20:06:44,500] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2017-11-06 20:06:44,501] - TelnetServer server is running on port 31000
  INFO root [2017-11-06 20:06:44,718] - ---- loginapp is running ----
 DEBUG root [2017-11-06 20:06:44,718] - Components::process(): Request for the process of identity...
  WARN root [2017-11-06 20:06:50,874] - Forwarding logs to logger[172.16.4.154:53266/0/0/0]...
 ERROR root [2017-11-06 20:10:48,596] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2017-11-06 20:10:48,597] - The following logs sent to logger failed:
  WARN root [2017-11-06 20:10:48,597] - ==> [2017-11-06 20:10:48 588] TCPPacketReceiver::processPendingEvents(172.16.4.154:53266): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2017-11-06 20:10:48,598] - ==> [2017-11-06 20:10:48 596] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [2017-11-06 20:10:48,800] - TCPPacketReceiver::processPendingEvents(172.16.4.154:53268): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2017-11-06 20:10:48,804] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [2017-11-06 20:10:48,816] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2017-11-06 20:10:48,817] - onLoginAppShutDown()
  INFO root [2017-11-06 20:10:48,817] - Shutdowner::onShutdown: shutting down
  INFO root [2017-11-06 20:10:48,817] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2017-11-06 20:10:48,817] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [2017-11-06 20:10:48,818] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [2017-11-06 20:10:48,818] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [2017-11-06 20:10:48,818] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [2017-11-06 20:10:48,818] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [2017-11-06 20:10:48,818] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [2017-11-06 20:10:48,818] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [2017-11-06 20:10:48,828] - Script::uninstall(): is successfully!
 DEBUG root [2017-11-06 20:10:48,828] - ThreadPool::destroy(): starting size 2.
  WARN root [2017-11-06 20:10:49,129] - ThreadPool::destroy(): waiting for thread(2)[0x785b88,0x7850f8,], try=1
 DEBUG root [2017-11-06 20:10:49,531] - TPThread::~TPThread(): 0x785b88
 DEBUG root [2017-11-06 20:10:49,531] - TPThread::~TPThread(): 0x7850f8
 DEBUG root [2017-11-06 20:10:49,531] - ThreadPool::destroy(): successfully!
 DEBUG root [2017-11-06 20:10:49,547] - Bundle::destroyObjPool(): size 13.
 DEBUG root [2017-11-06 20:10:49,547] - Channel::destroyObjPool(): size 15.
 DEBUG root [2017-11-06 20:10:49,547] - TCPPacket::destroyObjPool(): size 12.
 DEBUG root [2017-11-06 20:10:49,548] - UDPPacket::destroyObjPool(): size 2.
 DEBUG root [2017-11-06 20:10:49,548] - EndPoint::destroyObjPool(): size 14.
 DEBUG root [2017-11-06 20:10:49,548] - Address::destroyObjPool(): size 0.
 DEBUG root [2017-11-06 20:10:49,548] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [2017-11-06 20:10:49,548] - UDPPacketReceiver::destroyObjPool(): size 0.
  INFO root [2017-11-06 20:10:49,548] - loginapp(9000) has shut down.
  INFO root [2017-11-06 20:18:21,162] - 
  INFO root [2017-11-06 20:18:21,162] - -----------------------------------------------------------------------------------------


  INFO root [2017-11-06 20:18:21,163] - Resmgr::initialize: KBE_ROOT=C:/Users/hyc/Desktop/kbengine/
  INFO root [2017-11-06 20:18:21,163] - Resmgr::initialize: KBE_RES_PATH=C:/Users/hyc/Desktop/kbengine/kbe/res/;C:/Users/hyc/Desktop/kbengine/llgame/;C:/Users/hyc/Desktop/kbengine/llgame/scripts/;C:/Users/hyc/Desktop/kbengine/llgame/res/
  INFO root [2017-11-06 20:18:21,163] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/hyc/Desktop/kbengine/kbe/bin/server/
  INFO root [2017-11-06 20:18:21,165] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 172.16.4.154 ( 172.16.4.154:20013 )
  INFO root [2017-11-06 20:18:21,166] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 172.16.4.154:20013, SOMAXCONN=128.
  INFO root [2017-11-06 20:18:21,166] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 172.16.4.154 ( 172.16.4.154:53372 )
  INFO root [2017-11-06 20:18:21,167] - NetworkInterface::recreateListeningSocket(INTERNAL): address 172.16.4.154:53372, SOMAXCONN=128.
  INFO root [2017-11-06 20:18:21,168] - server-configs:
  INFO root [2017-11-06 20:18:21,168] - 	internalAddr : 172.16.4.154:53372
  INFO root [2017-11-06 20:18:21,168] - 	externalAddr : 172.16.4.154:20013
  INFO root [2017-11-06 20:18:21,168] - 	componentID : 9000
  INFO root [2017-11-06 20:18:21,553] - Components::findLogger: found logger, addr:172.16.4.154:53366
  INFO root [2017-11-06 20:18:21,553] - Components::addComponent: reset orderLog, uid=1001!
  INFO root [2017-11-06 20:18:21,553] - Components::addComponent[logger], uid=1001, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2017-11-06 20:18:21,557] - ---- loginapp Version: 0.9.12. ScriptVersion: 0.1.0. Protocol: 75C8DA776708D1E89693088467E5A4FB. Config: Debug X86. Built: 16:13:51 Jul  7 2017. AppID: 9000. UID: 1001. PID: 96344 ----
  INFO root [2017-11-06 20:18:21,558] - Server : Intel(R) Core(TM) i5-4590 CPU @ 3.30GHz with 4194303kB RAM
  INFO root [2017-11-06 20:18:21,558] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2017-11-06 20:18:21,558] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2017-11-06 20:18:21,684] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2017-11-06 20:18:21,684] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2017-11-06 20:18:21,685] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2017-11-06 20:18:21,686] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2017-11-06 20:18:21,686] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2017-11-06 20:18:21,687] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2017-11-06 20:18:21,719] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2017-11-06 20:18:21,749] - Script::install(): is successfully, Python=(3.4.2 (default, Jul  7 2017, 16:13:50) [MSC v.1800 32 bit (Intel)])!
  INFO root [2017-11-06 20:18:21,749] - Script::install(): KBExtra is successfully!
S_INFO root [2017-11-06 20:18:21,752] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2017-11-06 20:18:21,752] - TelnetServer server is running on port 31000
  INFO root [2017-11-06 20:18:21,965] - ---- loginapp is running ----
 DEBUG root [2017-11-06 20:18:21,965] - Components::process(): Request for the process of identity...
  WARN root [2017-11-06 20:18:28,168] - Forwarding logs to logger[172.16.4.154:53366/0/0/0]...
 ERROR root [2017-11-06 20:21:43,038] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2017-11-06 20:21:43,040] - The following logs sent to logger failed:
  WARN root [2017-11-06 20:21:43,040] - ==> [2017-11-06 20:21:43 038] TCPPacketReceiver::processPendingEvents(172.16.4.154:53366): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2017-11-06 20:21:43,040] - ==> [2017-11-06 20:21:43 039] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [2017-11-06 20:21:43,232] - TCPPacketReceiver::processPendingEvents(172.16.4.154:53367): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2017-11-06 20:21:43,232] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [2017-11-06 20:21:43,233] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2017-11-06 20:21:43,233] - onLoginAppShutDown()
  INFO root [2017-11-06 20:21:43,233] - Shutdowner::onShutdown: shutting down
  INFO root [2017-11-06 20:21:43,233] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2017-11-06 20:21:43,233] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [2017-11-06 20:21:43,233] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [2017-11-06 20:21:43,233] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [2017-11-06 20:21:43,233] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [2017-11-06 20:21:43,234] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [2017-11-06 20:21:43,234] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [2017-11-06 20:21:43,234] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [2017-11-06 20:21:43,245] - Script::uninstall(): is successfully!
 DEBUG root [2017-11-06 20:21:43,245] - ThreadPool::destroy(): starting size 2.
  WARN root [2017-11-06 20:21:43,547] - ThreadPool::destroy(): waiting for thread(2)[0x1fe54a0,0x1fe5508,], try=1
 DEBUG root [2017-11-06 20:21:43,948] - TPThread::~TPThread(): 0x1fe54a0
 DEBUG root [2017-11-06 20:21:43,948] - TPThread::~TPThread(): 0x1fe5508
 DEBUG root [2017-11-06 20:21:43,948] - ThreadPool::destroy(): successfully!
 DEBUG root [2017-11-06 20:21:43,956] - Bundle::destroyObjPool(): size 13.
 DEBUG root [2017-11-06 20:21:43,957] - Channel::destroyObjPool(): size 15.
 DEBUG root [2017-11-06 20:21:43,957] - TCPPacket::destroyObjPool(): size 12.
 DEBUG root [2017-11-06 20:21:43,957] - UDPPacket::destroyObjPool(): size 2.
 DEBUG root [2017-11-06 20:21:43,957] - EndPoint::destroyObjPool(): size 14.
 DEBUG root [2017-11-06 20:21:43,957] - Address::destroyObjPool(): size 0.
 DEBUG root [2017-11-06 20:21:43,957] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [2017-11-06 20:21:43,957] - UDPPacketReceiver::destroyObjPool(): size 0.
  INFO root [2017-11-06 20:21:43,957] - loginapp(9000) has shut down.
  INFO root [2017-11-06 20:21:48,462] - 
  INFO root [2017-11-06 20:21:48,462] - -----------------------------------------------------------------------------------------


  INFO root [2017-11-06 20:21:48,463] - Resmgr::initialize: KBE_ROOT=C:/Users/hyc/Desktop/kbengine/
  INFO root [2017-11-06 20:21:48,463] - Resmgr::initialize: KBE_RES_PATH=C:/Users/hyc/Desktop/kbengine/kbe/res/;C:/Users/hyc/Desktop/kbengine/llgame/;C:/Users/hyc/Desktop/kbengine/llgame/scripts/;C:/Users/hyc/Desktop/kbengine/llgame/res/
  INFO root [2017-11-06 20:21:48,463] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/hyc/Desktop/kbengine/kbe/bin/server/
  INFO root [2017-11-06 20:21:48,465] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 172.16.4.154 ( 172.16.4.154:20013 )
  INFO root [2017-11-06 20:21:48,466] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 172.16.4.154:20013, SOMAXCONN=128.
  INFO root [2017-11-06 20:21:48,466] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 172.16.4.154 ( 172.16.4.154:53420 )
  INFO root [2017-11-06 20:21:48,467] - NetworkInterface::recreateListeningSocket(INTERNAL): address 172.16.4.154:53420, SOMAXCONN=128.
  INFO root [2017-11-06 20:21:48,467] - server-configs:
  INFO root [2017-11-06 20:21:48,467] - 	internalAddr : 172.16.4.154:53420
  INFO root [2017-11-06 20:21:48,468] - 	externalAddr : 172.16.4.154:20013
  INFO root [2017-11-06 20:21:48,468] - 	componentID : 9000
  INFO root [2017-11-06 20:21:48,882] - Components::findLogger: found logger, addr:172.16.4.154:53417
  INFO root [2017-11-06 20:21:48,882] - Components::addComponent: reset orderLog, uid=1001!
  INFO root [2017-11-06 20:21:48,883] - Components::addComponent[logger], uid=1001, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2017-11-06 20:21:48,887] - ---- loginapp Version: 0.9.12. ScriptVersion: 0.1.0. Protocol: 75C8DA776708D1E89693088467E5A4FB. Config: Debug X86. Built: 16:13:51 Jul  7 2017. AppID: 9000. UID: 1001. PID: 97232 ----
  INFO root [2017-11-06 20:21:48,887] - Server : Intel(R) Core(TM) i5-4590 CPU @ 3.30GHz with 4194303kB RAM
  INFO root [2017-11-06 20:21:48,887] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2017-11-06 20:21:48,887] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2017-11-06 20:21:49,052] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2017-11-06 20:21:49,053] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2017-11-06 20:21:49,053] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2017-11-06 20:21:49,055] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2017-11-06 20:21:49,055] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2017-11-06 20:21:49,055] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2017-11-06 20:21:49,086] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2017-11-06 20:21:49,116] - Script::install(): is successfully, Python=(3.4.2 (default, Jul  7 2017, 16:13:50) [MSC v.1800 32 bit (Intel)])!
  INFO root [2017-11-06 20:21:49,116] - Script::install(): KBExtra is successfully!
S_INFO root [2017-11-06 20:21:49,118] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2017-11-06 20:21:49,118] - TelnetServer server is running on port 31000
  INFO root [2017-11-06 20:21:49,330] - ---- loginapp is running ----
 DEBUG root [2017-11-06 20:21:49,330] - Components::process(): Request for the process of identity...
  WARN root [2017-11-06 20:21:55,475] - Forwarding logs to logger[172.16.4.154:53417/0/0/0]...
 ERROR root [2017-11-06 20:26:06,707] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2017-11-06 20:26:06,709] - The following logs sent to logger failed:
  WARN root [2017-11-06 20:26:06,710] - ==> [2017-11-06 20:26:06 705] TCPPacketReceiver::processPendingEvents(172.16.4.154:53417): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2017-11-06 20:26:06,710] - ==> [2017-11-06 20:26:06 708] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [2017-11-06 20:26:06,921] - TCPPacketReceiver::processPendingEvents(172.16.4.154:53414): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2017-11-06 20:26:06,923] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [2017-11-06 20:26:06,927] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2017-11-06 20:26:06,928] - onLoginAppShutDown()
  INFO root [2017-11-06 20:26:06,929] - Shutdowner::onShutdown: shutting down
  INFO root [2017-11-06 20:26:06,929] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2017-11-06 20:26:06,929] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [2017-11-06 20:26:06,929] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [2017-11-06 20:26:06,929] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [2017-11-06 20:26:06,929] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [2017-11-06 20:26:06,929] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [2017-11-06 20:26:06,929] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [2017-11-06 20:26:06,929] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [2017-11-06 20:26:06,941] - Script::uninstall(): is successfully!
 DEBUG root [2017-11-06 20:26:06,941] - ThreadPool::destroy(): starting size 2.
  WARN root [2017-11-06 20:26:07,242] - ThreadPool::destroy(): waiting for thread(2)[0x1e62f40,0x1e63148,], try=1
 DEBUG root [2017-11-06 20:26:07,644] - TPThread::~TPThread(): 0x1e62f40
 DEBUG root [2017-11-06 20:26:07,644] - TPThread::~TPThread(): 0x1e63148
 DEBUG root [2017-11-06 20:26:07,644] - ThreadPool::destroy(): successfully!
 DEBUG root [2017-11-06 20:26:07,651] - Bundle::destroyObjPool(): size 13.
 DEBUG root [2017-11-06 20:26:07,651] - Channel::destroyObjPool(): size 15.
 DEBUG root [2017-11-06 20:26:07,652] - TCPPacket::destroyObjPool(): size 12.
 DEBUG root [2017-11-06 20:26:07,652] - UDPPacket::destroyObjPool(): size 2.
 DEBUG root [2017-11-06 20:26:07,652] - EndPoint::destroyObjPool(): size 14.
 DEBUG root [2017-11-06 20:26:07,652] - Address::destroyObjPool(): size 0.
 DEBUG root [2017-11-06 20:26:07,652] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [2017-11-06 20:26:07,652] - UDPPacketReceiver::destroyObjPool(): size 0.
  INFO root [2017-11-06 20:26:07,652] - loginapp(9000) has shut down.
  INFO root [2017-11-06 20:36:38,554] - 
  INFO root [2017-11-06 20:36:38,555] - -----------------------------------------------------------------------------------------


  INFO root [2017-11-06 20:36:38,555] - Resmgr::initialize: KBE_ROOT=C:/Users/hyc/Desktop/kbengine/
  INFO root [2017-11-06 20:36:38,556] - Resmgr::initialize: KBE_RES_PATH=C:/Users/hyc/Desktop/kbengine/kbe/res/;C:/Users/hyc/Desktop/kbengine/llgame/;C:/Users/hyc/Desktop/kbengine/llgame/scripts/;C:/Users/hyc/Desktop/kbengine/llgame/res/
  INFO root [2017-11-06 20:36:38,556] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/hyc/Desktop/kbengine/kbe/bin/server/
  INFO root [2017-11-06 20:36:38,557] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 172.16.4.154 ( 172.16.4.154:20013 )
  INFO root [2017-11-06 20:36:38,558] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 172.16.4.154:20013, SOMAXCONN=128.
  INFO root [2017-11-06 20:36:38,559] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 172.16.4.154 ( 172.16.4.154:53514 )
  INFO root [2017-11-06 20:36:38,560] - NetworkInterface::recreateListeningSocket(INTERNAL): address 172.16.4.154:53514, SOMAXCONN=128.
  INFO root [2017-11-06 20:36:38,560] - server-configs:
  INFO root [2017-11-06 20:36:38,560] - 	internalAddr : 172.16.4.154:53514
  INFO root [2017-11-06 20:36:38,560] - 	externalAddr : 172.16.4.154:20013
  INFO root [2017-11-06 20:36:38,560] - 	componentID : 9000
  INFO root [2017-11-06 20:36:39,000] - Components::findLogger: found logger, addr:172.16.4.154:53506
  INFO root [2017-11-06 20:36:39,000] - Components::addComponent: reset orderLog, uid=1001!
  INFO root [2017-11-06 20:36:39,001] - Components::addComponent[logger], uid=1001, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2017-11-06 20:36:39,005] - ---- loginapp Version: 0.9.12. ScriptVersion: 0.1.0. Protocol: 75C8DA776708D1E89693088467E5A4FB. Config: Debug X86. Built: 16:13:51 Jul  7 2017. AppID: 9000. UID: 1001. PID: 96444 ----
  INFO root [2017-11-06 20:36:39,005] - Server : Intel(R) Core(TM) i5-4590 CPU @ 3.30GHz with 4194303kB RAM
  INFO root [2017-11-06 20:36:39,006] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2017-11-06 20:36:39,006] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2017-11-06 20:36:39,133] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2017-11-06 20:36:39,133] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2017-11-06 20:36:39,134] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2017-11-06 20:36:39,135] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2017-11-06 20:36:39,135] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2017-11-06 20:36:39,136] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2017-11-06 20:36:39,168] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2017-11-06 20:36:39,200] - Script::install(): is successfully, Python=(3.4.2 (default, Jul  7 2017, 16:13:50) [MSC v.1800 32 bit (Intel)])!
  INFO root [2017-11-06 20:36:39,200] - Script::install(): KBExtra is successfully!
S_INFO root [2017-11-06 20:36:39,202] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2017-11-06 20:36:39,202] - TelnetServer server is running on port 31000
  INFO root [2017-11-06 20:36:39,418] - ---- loginapp is running ----
 DEBUG root [2017-11-06 20:36:39,418] - Components::process(): Request for the process of identity...
  WARN root [2017-11-06 20:36:45,590] - Forwarding logs to logger[172.16.4.154:53506/0/0/0]...
